%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Second Chapter *********************************
%*******************************************************************************

\chapter{Single molecule somatic mutation detection}

\ifpdf
    \graphicspath{{Chapter2/Figs/Raster/}{Chapter2/Figs/PDF/}{Chapter2/Figs/}}
\else
    \graphicspath{{Chapter2/Figs/Vector/}{Chapter2/Figs/}}
\fi

\section{Introduction}

Somatic mutations can occur in cells at all stages of life and in all tissues. The biochemical manifestation of a somatic mutation requires three distinct stages: DNA damage or modification from either endogenous or exogenous sources, mutation resulting from incorrect DNA damage repair and unrepaired DNA damage, and the persistence of the mutation in the genome of the cell and its descendants \cite{Stratton2009-of}. Most somatic mutations are benign, but some confer a proliferative advantage and are referred to as driver mutations. The advent of next-generation sequencing and the continued decline in sequencing costs have enabled us to sequence thousands of cancer genomes at scale and subsequent downstream sequence analysis has allowed us to discover tissue-specific driver mutations \cite{Martinez-Jimenez2020-kn}, identify biological processes that generate these mutations \cite{Alexandrov2013-kg}, to use somatic mutations as timestamps and biological barcodes to lineage trace development \cite{Behjati2014-gb}, to discover complex structural rearrangements such as chromothripsis \cite{Stephens2011-gj} that fundamentally changed the conventional view of tumorigenesis as the gradual process of the accumulation of somatic mutations \cite{Doll1954-of, Knudson1971-fg} and to better understand the relationship between abnormal embryonic development and paediatric tumour formation \cite{Marshall2014-ec}. International efforts such as the Cancer Genome Atlas (TCGA) program \cite{Weinstein2013-ko} and the Pan-Cancer Analysis of Whole Genomes (PCAWG) consortium \cite{ICGCTCGA_Pan-Cancer_Analysis_of_Whole_Genomes_Consortium2020-ts} have also measured and analysed genetic, epigenetic, transcriptomic and proteomic aberrations in thousands of tumour genomes to understand how these aberrations contribute to the hallmarks of cancer \cite{Hanahan2000-dp, Hanahan2011-zr}. 

Cancer is often described as the disease of the genome. Somatic mutation detection, hence, is often the first step towards characterising the cancer genome and these somatic mutations have been catalogued and analysed to determine their contribution to tumorigenesis. Multiple mutational process simultaneously acts on the genome at any given time and contributes to the accumulation of somatic mutations over an individual’s lifetime. To determine the mutational sources from a set of samples, mutational signature analysis is performed to either \textit{de novo} extract new mutational signatures or to assign the contribution of known mutational signatures to the mutation burden \cite{Alexandrov2013-fq}; a mutational signature is a mathematical abstraction of the likelihood that a particular biological process will produce a somatic mutation in a specific sequence context. During mutational signature analysis, somatic mutations are classified according to the event, the size of the event and the sequence context. Single base substitutions (SBS), for example, can be classified using the SBS96 classification system, which categorises SBS according to the six types of substitutions in the pyrimidine context (C>A, C>G, C>T, T>A, T>C and T>G) and the 16 possible trinucleotide sequence contexts derived from the 4 possible bases upstream and downstream of the substitution. SBS can be further subclassified based on their pentanucleotide sequence context (SBS1536 classification) and whether the SBS is located on the intergenic DNA, transcribed or untranscribed strand of the gene (SBS288 classification). Double base substitution, indel and structural variation classification system also exist for mutational signature analysis, but they are not the subject of interest in this chapter \cite{Alexandrov2013-fq, Li2020-vw, Steele2022-mn}. 

The PCAWG consortium has discovered 67 single-base-substitution (SBS) mutational signatures \cite{Alexandrov2020-ys}. To date, the biological aetiology for 49 SBS mutational signatures has been determined (Table X). The discovery of new somatic mutational signatures is an ongoing process where the number and the aetiology of mutational signatures is constantly updated and refined with increase in the number of sequenced genomes. Genomics England and collaborators, for example, have leveraged 100, 000 cancer genomes from around 85,000 patients to detect mutational signatures associated with rare and sporadic somatic mutagenesis \cite{Degasperi2022-qe}. In addition, somatic mutations resulting from chemotherapeutic agents is another active area of research \cite{Pich2019-ja, Aitken2020-sa}. Clinical sequencing of matched tumour and normal genomes is now routinely performed in the developed countries to help cancer patient treatment, fulfilling one of the many promises of the human genome project..

Somatic mutation detection, however, is not a solved problem. Somatic mutation callers, for example, employ different strategies and exhibit varying specificities and sensitivities. Consensus somatic mutation call, hence, is often used for downstream analysis \cite{Bailey2020-ou}. The base accuracy and read length, of Illumina reads, most importantly, is the common technical factor that limit the resolution at which the somatic mutations can be detected. MuTect, for example, cannot differentiate Illumina sequencing errors from low variant allele fraction (VAF) somatic mutations as a typical Illumina base call has a 0.01-1\% error rate \cite{Cibulskis2013-gw}. Library errors, introduced upstream of sequencing, is also often misclassified as somatic mutations \cite{Costello2013-cz, Chen2017-ba, Abascal2021-pk}. Newly acquired somatic mutations, therefore, are indistinguishable from background noise using conventional methods and required breakthroughs in sample and library preparation (Figure \ref{}). The detection of these somatic mutations, however, are critical for early detection of cancer, monitoring of tumour evolution during patient treatment and to enhance our understanding of the transformation of normal cells to neoplastic cells. 

The repeat content of the genome is another hurdle for accurate somatic mutation detection. Repetitive sequences (e.g., tandem repeat expansions, retrotransposons, segmental duplications, telomeric repeats and centromeric alpha-satellite) account for approximately 50\% of the human genome \cite{Lander2001-du}. If the repeat length is greater than the read length, read alignment software cannot determine the location of the read with respect to the reference genome as the read could have originated from any copies of the repetitive sequence \cite{Li2008-dt}. The accurate placement of reads, hence, requires repetitive sequences to be flanked with unique sequences not present elsewhere in the reference genome. Consequently, the reference genome is divided into callable region and non-callable regions based on mappability of Illumina short reads and variant calling is often restricted to the callable regions of the genome \cite{1000_Genomes_Project_Consortium2012-rj}. Clinically relevant genes in non-callable regions, hence, are often excluded from analysis \cite{Wagner2022-ph}.

The completeness and contiguity of the reference genome is often ignored, but another important factor for somatic mutation detection. The human reference genome constructed from physical mapping and clone-by-clone sequencing and assembly of overlapping BAC clones is undoubtedly the best mammalian reference genome \cite{Lander2001-du}, but the human reference genome is still incomplete. The human reference genome, for example, still has missing sequences, unplaced scaffolds and unlocalised scaffolds without a reference coordinate, and misassemblies such as incorrect sequence collapse and expansion. Furthermore, approximately 70\% of the human reference genome is derived from genomic DNA of an anonymous individual of African-European ancestry \cite{Osoegawa2001-np}. The current linear sequence of the human reference genome, therefore, may not accurately reflect the genomic diversity present in other populations and alternatively graph-based representation might better incorporate genomic diversity \cite{Garrison2018-ae}. The Genome Reference Consortium (GRC) has released grch38 build with alternative loci to address some of these issues \cite{Schneider2017-yo}. The recent completion of telomere-to-telomere CHM13 (T2T-CHM13) haploid genome using a combination of sequencing and mapping technologies has been a major milestone for genomics research \cite{Nurk2022-dv}. T2T-CHM13 genome, as expected, improve the accuracy and precision of both read alignment and variant calling \cite{Aganezov2022-dv}. 

Table of current somatic mutation callers, their sensitivity and specificity, and their approaches \cite{}.

Illumina’s technical limitations have limited somatic mutation detection to clonal or sub-clonal mutations. Two approaches have been developed to address these challenges: 1) to increase the copy number of the mutant DNA above the limit of detection threshold and 2) to increase the base accuracy of the Illumina reads through upstream changes in the library preparation protocol. Single-cell whole-genome amplification \cite{Lodato2018-hh}, single-cell clone expansion \cite{Lee-Six2018-qe} and laser-capture microdissection (LCM) \cite{Ellis2021-it} and sequencing adopts the former approach. Rolling circle amplification \cite{Lizardi1998-qh, Dahl2004-tm} and duplex sequencing (and its iterations) \cite{Schmitt2012-yr, Abascal2021-pk, Hoang2016-jx} adopt the latter approach where a highly accurate consensus sequence is created from multiple copies of a single molecule. Single-cell clone expansion and LCM sequencing are recognized as the gold-standard methods for somatic mutation detection in single-cells or clonal tissues, respectively. These methods have enabled the study of embryogenesis, somatic mutation rate, mutational processes, clonal structure, driver mutation landscape and earliest transformation of normal cells to neoplastic cells across a range of normal tissues, including adrenal gland, blood, bladder, bronchus, cardiac muscle, colon, endometrium, oesophagus, pancreas, placenta, prostate, skin, smooth muscle, testis, thyroid, ureter, visceral fat \cite{Lee-Six2018-qe, Martincorena2015-gu, Ju2017-vw, Martincorena2018-av, Brunner2019-xg, Lee-Six2019-vt, Yoshida2020-yr, Olafsson2020-vi, Moore2020-pi, Lawson2020-em, MSpencer_Chapman2021-cq, Coorens2021-ct, Robinson2021-te, Grossmann2021-gd, Moore2021-dl, Park2021-fx, Ng2021-jd}

Duplex sequencing, however, is the most scalable for option for ultra-rare somatic mutation detection and is the preferred method for circulating tumour DNA (ctDNA) based clinical applications \cite{Newman2016-cy}. Based on our understanding of duplex sequencing methods \cite{Schmitt2012-yr, Hoang2016-jx} and the recently developed nanorate sequencing protocol \cite{Abascal2021-pk}, a derivative of the duplex sequencing protocol and considering the similarities between two sequencing methods, we hypothesized that CCS reads might be as accurate or more accurate than duplex reads and that they can be used for single molecule somatic mutation detection (Figure \ref{}). The duplex library preparation protocol starts with the sonication and fragmentation of genomic DNA. Unique molecular identifier (UMI) consisting of 8 to 12 nucleotide and Illumina adapters are attached to double-stranded DNA molecules prior to their PCR amplification \cite{Schmitt2012-yr}. The duplex library is often diluted before PCR amplification to achieve optimal sampling and duplication per template molecule \cite{Hoang2016-jx, Abascal2021-pk}. PCR amplified library is sequenced and Illumina reads are subsequently grouped according to their UMI and are classified as Watson or Crick strand depending on whether the sequence was derived from Illumina adapter P5 or P7, respectively. A highly accurate double-strand consensus (duplex) sequence is generated leveraging the redundancies and complementarity between the forward and reverse strand reads. DNA polymerase, for example, might incorrectly replicate the template molecule during PCR amplification, but the polymerase error will be present only in one copy or a subset of the copies. In addition, non-complementary base pairing between the forward and reverse strand indicates the presence of polymerase errors. Consequently, duplex read promises theoretical base accuracy of $1 \times 10^{-9}$ (Q90), but in practice achieves base accuracy of $1 \times 10^{-6}$ (Q60) \cite{Schmitt2012-yr}

In contrast, duplex reads from the nanorate library protocol achieves the promised Q90 base accuracy \cite{Abascal2021-pk}. To accomplish this, the nanorate library protocol identifies and addresses library errors upstream of PCR amplification to produce duplex libraries from error-free native DNA molecules. Genomic DNA, for example, is fragmented not through sonication, but using a blunt end restriction enzyme to prevent enzymatic DNA misincorporation during end repair and gap-filling. Moreover, the addition of dideoxynucleotides also inhibits nick translation, rendering DNA molecules that require this process unsuitable for library creation. The use of the restriction enzyme currently limits somatic mutation detection to where the recognition sites are present.

PacBio CCS sequencing also takes advantage of the redundant sequencing and complementary base pairing between the forward and reverse strand to construct highly accurate consensus sequences. The single-strand reads are referred to as subreads and an individual subread has 10-15\% error rate \cite{Chaisson2012-vr}. CCS reads are reported to have an average read accuracy between Q20 and Q30, but their individual base accuracies have not been examined to date. We and others have hypothesized that PacBio circular consensus sequence (CCS) reads might be as accurate or more accurate than conventional duplex reads based on the similarities between the two protocols \cite{Wenger2019-pw}. PacBio CCS base quality score ranges from Q1 to nominal Q93, representing error rate of 1 in 5 billion bases. If the base quality score estimates are correct, we imagined that genome-wide single molecule somatic mutation detection will be possible across all human normal tissues, agnostic of clonality as the human genome accumulates 1 to 2 somatic mutation per human genome per 1-4 weeks. If successful, haplotype phased germline mutation (SNPs, indels and structural variations), 5-methylcytosine (5mC) and somatic mutation detection will be possible from bulk normal tissue CCS sequencing. Our imagination inspired us to examine single molecule somatic mutations where a single read alignment supports the mismatch between the read and the reference genome. Our understanding of somatic mutational processes across different tissue types was critical in selecting the samples to assess and demonstrate the potential for single molecule somatic mutation detection with PacBio CCS reads. 

Here, we invert the premise that long reads are inaccurate and propose that CCS reads have the highest base accuracy among commercially available sequencing platforms. We assess the potential for single molecule somatic mutation detection using PacBio CCS reads, identify systematic errors with consensus sequence generation and base quality score estimation and propose potential solutions to address these issues. In addition, we present himut, a method that can call somatic mutations where a single read alignment supports the mismatch between the sample and the reference genome. We detail the rationale behind the mechanics of himut and report its sensitivity and specificity. We have designed himut with ease of use in mind, and himut requires a sorted BAM file with primary read alignments as the only input and returns a VCF file with somatic mutations as output. Our software is publicly available at  https://github.com/sjin09/himut as a Python package under the MIT open license.

We selected a set of samples (BC-1, HT-115 and granulocytes from an 82-year-old female individual) as positive controls and a sample (cord blood granulocyte) with little or no somatic mutations as a negative control to determine the limit of detection, empirically calculate the PacBio CCS error rate and artefact signature resulting from systematic errors. In contrast to a typical sample where multiple mutational processes might be active at any given time, single-cell clone expansion and sequencing studies have definitively identified APOBEC, POLE, clock-like mutational processes to be the dominant ongoing somatic mutational processes in BC-1, HT-115 and granulocytes, respectively \cite{Petljak2019-wi, Mitchell2022-ry}. The mutational spectra from previous studies and the contribution of different mutational signatures to the mutational spectrum serves as truth sets to unbiasedly assess the accuracy of our somatic mutation detection algorithm and to experiment and evaluate the impact of different hard filters to sensitivity and specificity.

Single molecule somatic mutation candidates are generated from either a biological process or from a non-biological process such as library, sequencing, alignment, or systematic bioinformatics errors. Similarity between the mutational spectrum generated from the aggregate of somatic mutations and the expected mutational spectrum, hence, can be measured to assess the sensitivity of our method. In addition, mutational signature analysis can also be performed to determine the number of true negative and false negative somatic mutations from filtered somatic mutations to assess specificity. 

\section{Materials and Methods}

\subsection{CCS library preparation and sequencing}
BC-1 and HT-115 cell lines were cultured in XX media containing XX and at XX in a humidified X environment. Umbilical blood (PD47269d) and peripheral blood sample of an 82-year-old female individual (PD48473b) were collected in 40-60mL lithium-heparin tubes and blood granulocytes were subsequently isolated using Lymphophorep. High molecular weight (HMW) DNA from BC-1 and HT-115 cell line and PD47269d and PD484873b blood granulocytes were extracted using Qiagen MagAttract HMW DNA extraction kit () and was sheared to 16-20kb DNA fragments using Megaruptor 3 system () with speed setting X. CCS sequencing libraries were constructed according to the 0.9.0 CCS library preparation protocol () and the libraries were sequenced using Sequel IIe instrument at the Wellcome Sanger Institute. 

\subsection{CCS read alignment and germline mutation detection}
CCS reads with adapter sequences were identified with HiFiAdapterFilt \cite{Sim2022-pi} and were removed from downstream sequence analysis. CCS reads were aligned to the human reference genome (b37 and grch38) with minimap2 (version 2.24-r1155-dirty) with default parameters for CCS read alignment (-ax map-hifi --cs=short) \cite{Li2018-am} and primary alignments were selected, compressed, merged, and sorted with samtools (version 1.6) \cite{Li2009-qp}. Germline SNPs and indels were detected with deepvariant (version 1.1.0) \cite{Poplin2018-ub}. VCF files were compressed and indexed with tabix \cite{Li2011-zj} and left aligned and normalised with bcftools (version 1.17-7-g097bda6) \cite{Li2011-ag}

\subsection{CCS empirical base quality calculation}

To assess the potential for somatic mutation detection with CCS reads, we first assessed the accuracy of the BQ score estimate using CCS reads from cord blood granulocytes. The number of somatic mutations in cord blood granulocytes is limited to 40-50 somatic mutations per cell \cite{Osorio2018-mh}, and hence most SBS, excluding germline mutations, in cord blood granulocyte sample results from library, sequencing, alignment or bioinformatics error. The number of matches and mismatches were counted for each BQ score estimate to calculate the empirical BQ score. We considered reference allele and germline SNPs as matches and all other SBS as mismatches. Germline mutation detection using himut is described below. We excluded germline SNPs with genotype quality (GQ) score below minimum GQ score of 20 and read depth above maximum depth threshold $4d + \sqrt{d}$, where $d$ is the average read depth, from analysis. We, thereafter, calculated empirical BQ for each BQ score estimate (\ref{eq:1}): 

\begin{equation} \label{eq:1}
\text{empirical BQ} = -10\log_{10} \Big( \frac{\text{mismatch count}}{\text{match count}} \Big)
\end{equation}

To calculate the trinucleotide sequence context dependent CCS error rate, CCS reads from the cord blood sample were reconstructed, with the number of subreads for each CCS read set to 10 full-length subreads (the reasons are discussed in chapter 3). Cord blood CCS reads were subsequently processed as described above and below for read alignment and somatic mutation detection. To estimate the number of false positive mutations, the number of true positive somatic mutations were estimated from the number of callable bases and the cord blood somatic mutational process \cite{Mitchell2022-ry} and were subtracted from the number of trinucleotide sequence context normalised somatic mutation counts. The number of trinucleotide sequence context normalised false positive somatic mutation counts, and the number of callable trinucleotide bases were used to estimate the the trinucleotide sequence context dependent CCS error rate. 

\subsection{Germline and somatic mutation detection}

Germline and somatic mutations are both detected from bulk normal tissue a without matched normal leveraging CCS read length and base accuracy, characteristics unique to CCS reads and applicable hard filters from previous publications \cite{DePristo2011-vf, Kim2018-qi}. BAM file with sorted primary read alignments is the only required input to obtain a VCF file with somatic mutations. 



Upon initiation, read alignments are first randomly sampled from each target chromosome to compute the lower and upper bound read length and maximum read depth threshold $4d + \sqrt{d}$ where $d$ is the average read depth. SBS are collected from reads with average read accuracy, mapping quality score (MAPQ) and blast sequence identity greater than or equal to a predefined threshold. In addition, read length must be between the lower and upper bound read length to prevent somatic mutation detection from chimeric or fragmented reads. A naive Bayesian genotyper, thereafter, is applied to each SBS to determine whether the data $(D)$ only supports the germline mutation or whether the data supports both a germline mutation and a somatic mutation candidate simultaneously (\ref{eq:2}):

\begin{equation} \label{eq:2}
P(G|D) = \frac{P(G)P(D|G)}{P(D)} \smallskip
\end{equation}

where $P(G)$ is the prior probability of observing the germline mutation genotype and $D$ is the data that represents the pileup of read bases and corresponding sequencing error probabilities for each base at the substitution site. $P(D)$ is a constant across all the possible genotypes and is ignored. $P(G)$ is dependent on whether the genotype is heterozygous, heterozygous alternative (tri-allelic), homozygous alternative or homozygous reference allele with respect to the reference base (\ref{eq:3}):

\begin{equation} \label{eq:3}
 P(G)= 
 	\begin{dcases}
    	\theta & \text{if } G = g_{\text{het}} \\
	    \frac{\theta}{2} & \text{if } G = g_{\text{hetalt}} \\
		\theta^{2} & \text{if } G = g_{\text{homalt}} \\
		1 - \frac{3\theta}{2} - \theta^{2} & \text{if } G = g_{\text{homref}} \\
	\end{dcases} \smallskip
\end{equation}

where $\theta$ is the expected germline SNP frequency and the default $\theta$ is set as $1\times10^{-3}$, the expected human germline SNP frequency. 

$P(D|G)$ is the probability of observing the data given the genotype. Binomial likelihood is calculated for each genotype under the assumption that sequencing errors and read sampling is independent and identically distributed (\ref{eq:4}):

\begin{equation} \label{eq:4}
P(D|G) =  
	\begin{dcases}
    	\frac{1}{2^n}\prod_{i}^{n} P(b_{i}|G) & \text{if } G = g_{\text{het}} \text{ or } g_{\text{hetalt}} \\
	    \prod_{i}^{n} P(b_{i}|G) & \text{if } G = g_{\text{homalt}} \text{ or } g_{\text{homref}} \\
	\end{dcases}
\end{equation}

where $P(b|G)$ is the probability of observing the base given the genotype and is defined as such \ref{eq:5}

\begin{equation} \label{eq:5}
P(b_{i}|G) = P(b_{i}|A) = 
	\begin{dcases}
    	1 - \epsilon_{i} & \text{if } b_{i} \in A \\
	    \frac{\epsilon_{i}}{3} & \text{if } b_{i} \not\in A \\
	\end{dcases} \smallskip
\end{equation}

where $b$ is CCS base covering the target locus, $\epsilon$ is the corresponding sequencing error probability and $A$ is allele of the genotype. In practice, all calculations are performed in log scale. Phred scaled likelihood (PL) is calculated for the 10 possible genotypes (AA, CA, CC, CT, GA, GC, GG, GT, TA, TT) using the posterior probability of the genotype (\ref{eq:6}):

\begin{equation} \label{eq:6}
\text{PL} = -10\log_{10}P(G|D) 
\end{equation}

and PL for each genotype is normalised using the lowest PL (\ref{eq:7}).

\begin{equation} \label{eq:7}
\text{normalised PL} = [\text{PL}_{i}, \text{PL}_{i+1}, \ldots, \text{PL}_{10}] - \text{PL}_{i}
\end{equation}

where PL is assumed to be sorted from the smallest to the largest. The genotype with the lowest PL is selected as the germline genotype. Genotype quality (GQ) score of the selected germline genotype is the difference between the second lowest normalised PL and the lowest normalised PL. If the data only provides evidence for a germline mutation, next SBS is then considered for somatic mutation detection. If the data supports the presence of both a germline mutation and a somatic mutation candidate, a number of conservative hard filters are applied to distinguish somatic mutations from errors:

\begin{enumerate}
\item If the germline mutation is a heterozygous, heterozygous alternative or homozygous alternative allele, somatic mutation candidate is excluded from the downstream analysis as somatic reversions are not considered. Somatic mutation detection, hence, is restricted to locus with homozygous reference allele such that heterozygous mutation is not misclassified as a somatic mutation.
\item The GQ score for the homozygous reference allele needs to be above the minimum GQ score threshold.
\item The BQ score of the somatic mutation candidate needs to be above the minimum BQ score threshold.
\item Indels must be absent from the target locus.
\item The read depth of the target locus needs to below the maximum depth threshold.
\item The reference allele count and the alternative allele count needs to be above the minimum reference allele and alternative allele count. This condition is not required if the sample has sufficient sequence coverage as the GQ score is positively correlated with sequence coverage.
\item CCS reads with adapter sequences might still be present in the BAM file and somatic mutation candidate might result from incomplete adapter trimming. Candidates located in close proximity to start and ends of reads are filtered as specified with the -{}-min\_trim parameter. 
\item The number of mismatches adjacent to the candidate needs to be below the -{}-max\_mismatch\_count threshold as an alignment error can be mistaken as a somatic mutation. The size of the window is specified with the -{}-mismatch\_window parameter. 
\end{enumerate}

A VCF file with common SNPs (1\%>major allele frequencies) and a Panel of Normal (PoN) VCF file can also be optionally provided to exclude somatic mutation candidates potentially resulting from DNA contamination and systematic bioinformatics error, respectively. In addition, a VCF file with haplotype phased hetSNPs can be provided to limit somatic mutation detection from haplotype phased CCS reads. Here, himut with default parameters (-{}-min\_qv 30 -{}-min\_sequence\_identity 0.99 -{}-min\_gq 20 -{}-min\_bq 93 -{}-min\_trim 0.01 -{}-min\_ref\_count 3 -{}-min\_alt\_count 1 -{}-min\_hap\_count 3 -{}-mismatch\_window 20 -{}-max\_mismatch\_count 0) were used for the identification of unphased and haplotype phased somatic mutation. As sex chromosomes are enriched for misassembled regions and repetitive sequences, somatic mutation detection was restricted to the autosomes. To process BAM, FASTA/Q and VCF files, himut internally uses pysam \cite{pysam}, pyfastx \cite{Du2021-ya} and cyvcf2 \cite{Pedersen2017-ld}, respectively. In addition, multiprocessing Python package is used to enable parallel processing of each chromosomes. 

\subsection{Panel of Normal construction}

We created a PoN VCF file from 11 normal individuals with publicly available CCS dataset (Table X) to reduce number of false positives arising from systematic bioinformatics errors. We ran himut with relaxed parameters (-{}-min\_mapq 30 –{}-min\_trim 0 –{}-min\_sequence\_identity = 0.8 –{}-min\_hq\_base\_proportion 0.3 –{}-min\_alignment\_proportion 0.5 –{}-min\_bq = 20) to maximise the number of mutations called from these samples. The number of samples in the PoN VCF is currently limited to the number of publicly available CCS dataset. As the number of CCS sequenced samples increases, in the future the power to distinguish somatic mutations from artefacts will also increase.  

\subsection{Germline mutation haplotype phasing}

Haplotype phasing requires one to determine whether the polymorphisms are derived from a contiguous set of mutations. We treat haplotype phasing as a graph algorithms problem where each hetSNP is a node and measure haplotype consistency between a pair of hetSNPS to determine the validity of the edge. A single CCS read can span multiple heterozygous SNPs (hetSNPs) and a set of CCS reads can be used to measure the haplotype consistency between a pair of hetSNPs. Haplotype consistency if measured between all pairwise hetSNP and a pair of hetSNP is determined to be haplotype consistent through a binomial test (p<0.0001, one-sided). If a hetSNP is haplotype consistent with at least 20\% of its possible pairs, hetSNP is a haplotype consistent hetSNP. Using the breadth-first-search algorithm, haplotype consistent hetSNPS are connected to construct a haplotype block and both haplotype consistent and haplotype inconsistent hetSNPs are returned as a VCF file. 

\subsection{Haplotype phased somatic mutation detection}

CCS reads are typically phased using adjacent hetSNPs. CCS reads, however, spans multiple hetSNPs and can be used to construct haplotype blocks. We use CCS reads to construct haplotype blocks (discussed below) and assign CCS reads to haplotype blocks. If the CCS read belongs to two haplotype blocks or if the hetSNPs belonging to the CCS read doesn’t match the haplotype phased hetSNPs exactly, CCS read is determined to be not phased. In addition, a hetSNP can be misclassified as a somatic mutation if the two haplotypes are sampled unevenly and hence we require both h0 and h1 haplotype counts of the wild type CCS reads without the somatic mutation in the region to be above the --min\_hap\_count 3. 

\subsection{CCS read base quality score estimation and recalibration}

BAMsieve [ref, github] was used to select subreads where a productive ZMW created a CCS read with average read accuracy above Q20. abPOA \cite{Gao2021-nf} was used to construct partial order alignments between CCS and subreads from the same ZMW and the partial order alignments were parsed to select CCS bases where there was unanimous support from all the subread bases. The CCS bases with unanimous support was assigned Q93 base and all the other bases were assigned Q0 base and himut was used to call somatic mutations from CCS reads with recalibrated base quality scores. 

XXX was used to align subreads to CCS reads from the same ZMW [ref, github] and samtools was used to compress the alignments and to select primary alignments. DeepConsensus (version --, command: ) \cite{Baid2022-or} takes as input the BAM file with subreads aligned to the CCS reads and returns polished CCS reads with recalibrated BQ scores. Himut was used to call somatic mutations from DeepConsensus polished CCS reads. 

\subsection{Single-base-substitution count normalisation}
To determine the correct number of substitutions called per genome, the number of CCS bases where the substitution could have been detected from has to be determined considering the trinucleotide context frequencies in the reference genome.

\begin{equation}
f_{i} = \frac{t_{i}}{\sum^{32}_{i=1} t_{i}}
\end{equation}

\begin{equation}
r^{\text{callable}}_{i} = \frac{f^{g_{\text{callable}}}_{i}}{f^{\text{CCS}_{\text{callable}}}_{i}}
\end{equation}

\begin{equation}
r^{g}_{i} = \frac{f^{g_{\text{callable}}}_{i}}{f^{g}_{i}}
\end{equation}

\begin{equation}
S'_{\text{ACA>A}} = S_{\text{ACA>A}} \times r^{\text{callable}}_{\text{ACA}} \times r^{g}_{\text{ACA}}
\end{equation}

\begin{equation}
m_{\text{ACA}} = \frac{S'_{\text{ACA>C}} + S'_{\text{ACA>G}} + S'_{\text{ACA>T}}}{t^{\text{CCS}_{\text{callable}}}_{\text{ACA}}} 
\end{equation}

\begin{equation}
g_{\text{burden}} = \sum^{32}_{i=1} m_{i} * t^{g}_{i}
\end{equation}

%\times r^{g}_{\text{ATG}}


%\begin{equation}
%r^{g}_{i} = \frac{f^{g_{\text{callable}}}_{i}}{f^{g}_{i}}
%\end{equation}

We apply the same conditions as somatic mutation detection to all the CCS reads with and without the somatic mutation, determine the trinucleotide sequence context count from all the CCS bases where the same conditions would have been applied, calculate the ratio of trinucleotide sequence context frequency between the reference genome and the CCS bases. The single base substitution count is multiplied by the trinucleotide sequence context ratio to calculate the normalised single base substitution count. The normalised SBS count is used to calculate the mutation burden and to generate the mutational pattern plots. 

 \section{Results}

\subsection{CCS library errors and sequencing errors}

CCS reads have been successfully used for construction of highly contiguous and complete de novo assemblies \cite{} and germline mutation detection \cite{}. In these applications, the accuracy of individual base quality scores is not as important as ~50\% or ~100\% of the bases will support the consensus base, heterozygous or homozygous mutation. The accuracy of individual base quality scores, however, matters for ultra-rare somatic mutation detection as the base accuracy must be higher than the human genome somatic mutation rate (1-2 mutations per 1-4 weeks per cell), equivalent to approximately ~Q90 to distinguish sequencing errors from single molecule somatic mutations. In addition, library, sequencing and systematic errors and genomic DNA contamination are common sources of false positive somatic mutations. 

We generated 30-fold CCS sequence coverage from BC-1, HT-115 and blood granulocytes from an 82-year-old female individual (PD48473b) and 70-fold CCS sequence coverage from cord blood granulocyte (PD47269d) with an average read length between 16 and 20kb (Table \ref{tab:CCS-sequence-statistics}) to achieve these objectives: 1), assess the potential for single molecule somatic mutation detection with CCS reads, 2) identify and address the sources of errors where possible and 3) empirically estimate the PacBio CCS error rate to define the limit of detection threshold, 4) develop a method for somatic mutation using CCS reads and 5) assess the sensitivity and specificity of our method.

\begin{table}[h]
\caption{Experimental Data}
\label{tab:CCS-sequence-statistics}
\begin{adjustbox}{max width=1.1\textwidth,center}
\begin{tabular}{l|cccc}
                                     & BC-1 & HT-115 & PD47269d & PD4873b \\ \hline
Genomic DNA source                   & \multicolumn{2}{c}{Cell line} & \multicolumn{2}{c}{Blood granulocyte} \\  \hline
Age (years)                 		 & - & - & 0 & 82  \\ \hline
CCS read count                       &  5,962,252 &  5,933,281 & 12,156,251 & 4,949,180 \\ \hline
Mean length $\pm$ std (bp)  & 18,571 $\pm$     & 17,038 $\pm$   &  16,523 $\pm$ 3,752 & 18,263 $\pm$ 1,753 \\ \hline
Q93 bases (\%) 						 & 51.4 & 55.5 & 57.6 & 51.7 \\ \hline
Sequence coverage 				     & 36.9 & 33.7 & 67.0 & 30.1 \\ \hline
Mutational process   			     & APOBEC & POLE & \multicolumn{2}{c}{Clock-like} \\ \hline
Mutational signature 				 & SBS2   & SBS10a, SBS10b and SBS28 & \multicolumn{2}{c}{SBS1 and SBS5} \\ \hline
Mutation burden per cell 		     & $\sim$2,000 - 22,000 & $\sim$8,000 - 11,000 & $\sim$40 - 50 & $\sim$1400 - 1500 \\ \hline 
\end{tabular}
\end{adjustbox} 
%x\floatfoot{\small{CCS sequencing statistics, mutational process, associated mutational signatures and mutation burden are described for the negative control (PD47269d) and positive control (BC-1, HT-115 and PD48473b) samples.}}
\end{table}

We, first, examined the library preparation and circular consensus sequence construction process to minimise the number of library and sequencing errors. HMW DNA for CCS library preparation is often prepared through Qiagen Magattract or Circulomics HMW DNA extraction kit and HMW DNA is sheared to the appropriate size using a Megaruptor instrument. A hairpin adapter is attached to both ends of the double-stranded DNA molecule to create a topologically circular template. DNA nuclease is subsequently used to digest DNA molecules (e.g, failed ligation products) not suitable for sequencing. Primer with poly-A tail, thereafter, is annealed to the hairpin adapter sequence. BluePippin based size selection may additionally be performed to prepare size-selected libraries to maximize sequence throughput per SMRTcell. 

A DNA damage repair enzyme cocktail (unpublished) is used to repair DNA damage (nicks, abasic sites, thymidine dimers, blocked 3’-ends, oxidised guanine and pyrimidines and deaminated cytosines) introduced during library preparation (personal communication). In addition, end-repair and A-tailing is performed to remove protruding ends and to enable adapter ligation, respectively. Defective DNA damage repair or unrepaired DNA damage manifest as library errors and can be misclassified as a somatic mutation. The precise identity of DNA damage repair enzymes in the cocktail are unknown. We, however, can make informed assumptions about their function and their impact on downstream sequence analysis, and highlight the DNA damage repair process that is most likely to introduce library errors. Nanoseq protocol, for example, pinpoints end-repair and nick translation process to be the primary sources of library errors. Strand-displacement synthesis during nick translation, for example, can introduce kilobases of sequences using the complementary strand as a template (Figure \ref{}) \cite{}. 

CCS libraries are loaded on the SMRTcell and template DNA molecules diffuses into one of the ZMWs. A productive ZMW is defined as a ZMW with a single template molecule, from which a sufficient number of subreads are sequenced to construct a consensus sequence with at least Q20 average read accuracy. DNAP at the bottom of the ZMW binds to the DNA primer and initiates rolling circle amplification through strand-displacement synthesis. DNAP incorporates fluorescently labelled nucleotides, fluorescence emitted during DNA incorporation is measured and fluorophore is cleaved off upon successful incorporation. The wavelength of the fluorescence, length of the fluorescence, and duration between the successive pulses of fluorescence is used to determine the identity of the base and chemical modifications to the base. 

The DNAP from the latest library protocol has sufficient processivity to generate an average of 10-12 full-length subreads on average for template molecule with read-of-insert length between 16kb and 20kb. The single-strand readouts of the forward and reverse strand of the template molecule are referred to as subreads. The first subread and the last subread are often partial readouts of the template molecule because of internal priming and sequencing termination, respectively while the subreads from the second to the second-to-last subreads are full-length readouts of the template molecule (Figure \ref{}).  Assuming perfect detection of adapter sequences, odd-numbered subreads and even-numbered subreads are assumed to have the same sequence orientation as DNAP is agnostic to strand orientation. A draft sequence is constructed from multiple sequence alignment of subreads and is polished based on the realignment of subreads to the draft sequence. A dinucleotide sequence context Hidden Markov Model (HMM) is used to infer the base accuracy and DNA sequence from the observed subread bases (personal communication). A highly accurate consensus sequence can be constructed as sequencing errors are assumed to be randomly introduced without sequence context bias and are independent of one another. In addition, non-complementary base pairing between the forward and reverse strand indicates the presence of either a library or a sequencing error and resulting CCS is assigned a low BQ score. 

PacBio circular consensus sequence algorithm (pbccs) calculates the median subread length and uses subreads with read length above 50\% of median subread length and below 200\% of median subread length for CCS generation for CCS generation. If adapter sequences are incorrectly detected within the subread or if adapter sequences are not detected where present, full-length subreads can be fragmented into multiple partial subreads and multiple subreads can be concatenated into a single subread, respectively. Unfortunately, read length based hard filters cannot identify all cases where adapter sequence detection has failed. 

To identify potential errors introduced during CCS library preparation and sequencing, CCS and subreads from the same ZMW were analysed together and sequence quality control was performed (Methods). We observed that X\% of ZMWs have fragmented and/or concatenated subreads (Figure \ref{}). We hypothesise that CCS reads with read length deviating from mean CCS read length are the result of failed adapter sequence detection and exclude these CCS reads from somatic mutation detection (Method). In addition, we also noticed higher than expected adenine and thymine proportion at the end of CCS reads resulting from incomplete adapter sequence trimming (Figure \ref{}) 

% CCS BQ score is biased towards Q93. 

CCS reads have an average read accuracy of at least Q20 and individual BQ score ranges from Q1 to nominal Q93, corresponding to $0.5\times10^{-9}$ error rate (Figure \ref{}). To our knowledge, the accuracy of CCS BQ has not been examined to date. CCS read accuracy and BQ score is dependent on the number of subreads per CCS read (Figure \ref{}) and concordance between the subread bases and the CCS base. We confirm that the number of substitutions and indels is negatively correlated with CCS read accuracy and the number of subreads per CCS read as reported in a previous publication (Figure \ref{}). The accuracy of the BQ score, hence, is expected to increase with the number of supporting subread bases. We, however, observed that the accuracy of the CCS BQ score decreases with increase in the number of subreads and that increase in the number of subreads per CCS read results in not diminishing returns, but negative returns to CCS base accuracy (discussed later in Chapter 3). To determine whether CCS bases have sufficient base accuracy for single molecule somatic mutation detection, we measured the empirical BQ score using cord blood CCS reads and (Methods) and ascertained that CCS bases have sufficient accuracy for rare somatic mutation detection where a sample has a high mutation burden or a high somatic mutation rate (Figure \ref{}). Using positive control samples, we identified additional CCS read characteristics that influences somatic mutation detection sensitivity and specificity. 

\subsection{Germline mutation and somatic mutation detection}

Somatic mutagenesis is a continuous process throughout life. Bulk normal tissue has germline mutations that are inherited from parents, mosaic mutations that occurred during embryonic development and newly acquired somatic mutations from ongoing mutational processes. In addition, cells with driver mutations can outcompete neighbouring cells and undergo clonal expansion. Paired tumour-normal sequencing is often performed to distinguish germline mutations from somatic mutations in a tumour sample. Here, we present how we distinguish errors and germline mutations from somatic mutations in bulk normal tissue, leveraging CCS read length and base accuracy. 

We, first, compared germline SNPs detected from both himut and deepvariant to assess whether our algorithm can accurately call genetic variations (Table). The number of SNPs and transition to transversion (TiTv) ratio is within the expected range, demonstrating that himut can also function as a standalone variant caller. We believe that algorithmic differences account for disparities in number of SNPs called with himut and deepvariant, which is a deep learning based variant caller that uses read pileup images for germline mutation detection while himut uses an analytical approach similar to GATK for germline mutation detection. 

To distinguish germline mutations from somatic mutations, himut detects and classifies germline mutations as heterozygous, heterozygous alternative, homozygous alternative, or homozygous reference allele (Method, Figure \ref{}). Somatic mutation candidates are collected from CCS reads meeting the defined read-level prerequisites and candidates are categorised according to their base-level conditions (Figure \ref{}). Somatic mutation detection is also restricted to homozygous reference allele sites as somatic reversions might be the result of DNA contamination. To calculate the mutation burden of the sample, himut identifies the number of callable bases using the same conditions as somatic mutation detection and normalises the somatic mutation count based on the number of callable CCS bases and reference bases (Method). A VCF file with haplotype phased heterozygous SNPs (hetSNPs), a VCF file with common SNPs and a PoN VCF file can also be optionally provided to call haplotype phased somatic mutations, to exclude false positive mutations resulting from DNA contamination and discard false positive mutations arising from systematic errors, respectively. 

CCS read length and base accuracy can also be leveraged to phase hetSNPs and construct contiguous haplotype blocks, which enables haplotype phasing of CCS reads and haplotyped phased somatic mutation detection (Method). Read-backed phasing with Illumina reads uses adjacent hetSNPs to phase approximately $\sim$30\% of detected somatic mutations \cite{}. In contrast, haplotype phased somatic mutation detection with CCS reads uses all hetSNPs that CCS read spans and phases approximately $\sim$ 70\% of somatic mutations (Figure \ref{}). In addition, haplotype phased somatic mutation detection has three advantages: 1) CCS reads derived from DNA contamination often do not possess the same haplotype as the sample. If CCS read do not share the consensus haplotype, CCS read is excluded from somatic mutation detection (Figure, 2\ref{}) If two haplotypes are unevenly sampled, hetSNP can be misclassified as somatic mutations in low coverage samples. Restricting somatic mutation detection to haplotype phased regions limits somatic mutation detection to regions where both haplotypes have been adequately sampled. (Figure \ref{}), 3) CCS read with the same somatic mutations should share the haplotype and somatic mutation should not be present on both haplotypes (Figure \ref{}). Haplotype phased somatic mutation detection is especially helpful for samples with high heterozygosity.

\subsection{Somatic mutation detection sensitivity and specificity}

We called and benchmarked haplotype phased and unphased somatic mutations from the three positive controls with different mutational burdens and distinct mutational processes. Our unique benchmarking approach leverages the fact that a single somatic mutational process is active in each sample and that somatic mutation candidates are derived from either errors or newly acquired somatic mutations. We cannot be certain whether individual somatic mutations are derived from a biological process or a non-biological process, but mutational spectrum produced from the aggregate somatic mutations should be consistent with the expected mutational signature, if there is sufficient signal-to-noise ratio for somatic mutation detection. In addition, our approach is not biased towards Illumina callable regions of the genome unlike the Genome in a Bottle (GIAB) benchmarks \cite{} as our somatic mutation detection method is agnostic to reference position. 

We calculated mutation burden from BC-1, HT-115 and PD48473b sample to be X, X, X, respectively, consistent with previous estimates \cite{}. In addition, high cosine similarity between the expected mutational signatures and mutational spectrum from our positive control samples demonstrate that PacBio CCS bases have sufficient base accuracy for rare somatic mutation detection where samples have a high mutation burden or high somatic mutation rate (Method). Moreover, we can determine the number of true positive mutations and false positive mutations from the called somatic mutations and the number of true negative mutations and false negative mutations from the filtered somatic mutations through mutational signature analysis. We can subsequently use these estimates to calculate the sensitivity, specificity, specificity, and F1-score for each of our sample (Method, Table). We also selected appropriate hard filter thresholds based on receiver operating characteristic (ROC) curve generated under a range of hard filter conditions (Figure \ref{}) and determined hard filters with the greatest impact on sensitivity based on odds ratio calculated in the absence and presence of the hard filter in question. BQ and GQ score were crucial for somatic mutation detection while other filters had marginally positive impact on somatic mutation sensitivity. We would like to also highlight that somatic mutation detection sensitivity and specificity increased when grch38 was used as a reference genome, reflecting better representation of genetic polymorphisms with improvements in assembly quality (Table). We, unfortunately, could not compare himut with other methods as himut is the first somatic SBS detection method with CCS reads and as somatic mutation detection below 0.1\% VAF has not been technically feasible with Illumina reads. 

\subsection{CCS errors, error rate calculation and base quality score recalibration}

The mutation burden in cord blood sample is the lowest, with only 40-50 somatic mutations per cell \cite{}. CCS bases, unfortunately, does not have sufficient signal-to-noise ratio to enable somatic mutation detection in cord blood sample with high confidence. Mutational spectrum from the cord blood sample, which we refer to as the CCS error profile, is dissimilar to the expected mutational signature as the number of false positive mutations exceeds the number of true positive mutations (Figure \ref{}). CCS error profile occurs in multiple samples, suggesting that the error process is systematic in nature (Figure \ref{}). Using the number of false positive mutations and the callable number of bases, we calculated the CCS error rate to range from Q60 to Q90 depending on the substitution and the trinucleotide sequence context (Method, Figure \ref{}). 

Library, sequencing, and software error upstream of somatic mutation detection are potential sources of false positive mutations. We triangulated software error as the origin of the CCS error profile through somatic mutation detection using uncapped BQ scores, deepConsensus polished CCS reads \cite{} and CCS reads with recalibrated BQ scores (Method, Figure \ref{}). 

CCS BQ score ranges from Q1 to Q93 and BQ scores are encoded with the ASCII character encoding format. BQ score is capped at Q93 because ASCII character cannot support Phred-scaled quality values (QV) above 93. Inability to detect somatic mutations accurately with uncapped BQ scores demonstrates that there is a persistent problem with BQ score estimation (Figure \ref{}). 

DeepConsensus calculates BQ score based on alignment of subreads to the CCS read from the same ZMW and BQ score of deepConsensus polished CCS reads ranges from Q1 to Q50 (Figure \ref{}), which we think is too conservative considering the empirical BQ score estimation from the cord blood sample. We also observed that somatic mutation detection with polished Q50 CCS bases did not generate the expected mutational spectrum while that with polished CCS bases with BQ score above Q30 generated the expected mutational spectrum, suggesting that once again BQ score is not accurately estimated. 

To assess potential for single molecule somatic mutation detection with CCS reads, we performed partial order alignment between CCS read and subreads from the same ZMW and identifies bases where there is unanimous support for the CCS base from the subreads (Method). Somatic mutation detection with CCS bases with unanimous support from subreads generate mutational spectrum expected from the cord blood sample, suggesting that software error and not sequencing error is the source of false positive mutations. We hypothesise that consensus sequence construction and polishing algorithm consider somatic mutation as errors and as a result have incorrect sequencing error priors and BQ score estimates. 

\section{Conclusion}

Here, we demonstrate that a subset of PacBio CCS has sufficient base accuracy to enable single molecule somatic SBS detection.

We estimate that CCS base accuracy ranges from Q60 to Q90 depending on the substitution and the trinucleotide sequence context. The CCS error rate is unexpectedly also dependent on the average number of supporting of subreads per CCS read (discussed in Chapter 3). The false positive substitutions resulting from inaccurate BQ scores are shared across samples and sequencing runs, suggesting that the issue is systematic in nature. Using a modified pbccs that returns uncapped BQ scores, we have confirmed that the same issue extends to CCS bases with BQ score above Q93. Google has developed deepConsensus to polish CCS bases and to revise CCS BQ scores based on multiple sequence alignments between subreads and CCS read from the same ZMW [ref]. deepConsensus BQ score estimates is capped at Q50, which is too conservative in comparison to our empirical calculation and similarly inaccurate as single molecule somatic mutation detection is not possible with deepConsensus Q50 CCS bases. We hypothesize the conservative deepConsensus BQ score estimate is since kmers arising from somatic mutations are treated as errors. 

We observed that the false positive substitution is identical to the 5’ and 3’ and potentially the false positive substitution arises from the fact pbccs uses dinucleotide sequence context HMM and potentially a trinucleotide sequence context HMM might address the issue. 

\section{Discussion}

To date CCS reads have ben successfully used for germline SNP, indel and structural variation detection and have improved the genetic diagnosis rate of previously undiagnosed rare diseases [ref, ref, Chaisson and Eichler, ngmlr, sniffles, deepvariant]. In addition, assemblies in combination with strand-seq enable detection of haplotype phased structural rearrangements longer than the read length [ref]. The applications of CCS read for somatic mutation detection, however, have been limited to date. Others have had limited success in using long reads for studying complex structural rearrangements in cancers and somatic retrotransposition detection [ref, ref]. The ability to detect large scale somatic structural rearrangements with long reads is especially important in determining the combination of genomic changes that results in the somatic structural variation. Here, we have focused on the successful detection of somatic SBS, but the method could be potentially improved to somatic indel detection. The somatic mutations detected from our approach are not all true somatic mutations and if a user wishes to determine the confidence of the somatic mutation call or determine the posterior probability of the somatic mutation call, user can calculate the posterior probability of the substitution coming from a specific trinucleotide sequence context to have been generated by a specific and known mutational signatures [,ref, Eq]. In the future, when the CCS base quality scores are properly calibrated, single molecule somatic mutation detection might be truly possible.

Here, we did not focus on identifying and addressing the CCS library errors. We, however, believe that library errors must be present in CCS reads. HMW DNA shearing using XXX, for example, introduces oxidative DNA damage. 5’ filling or 3’ filling with XXX enzymes can perform strand displacement and use the template strand to synthesize the complementary strand, and these processes have been documented to generate library errors (ref, Nanoseq). To eliminate the library errors, HMW DNA could potentially be obtained from blunt-end restriction enzyme digestion, perform A-tailing and hairpin adapters could be ligated through blunt-end ligase. In addition, DNA molecules dependent on strand displacement and synthesis can be made not-viable for library preparation with the addition of dideoxy nucleotides or with DNA restriction enzymes that digests single-strand DNA. 

PacBio CCS bases are at least hundred thousand-fold to one million-fold more accurate than Illumina short read bases. 

Our method and CCS sequencing can be used to identify the presence of MMR for immunotherapy purposes. 

In addition, the method is focused on somatic mutation detection from normal tissues but can be extended to matched tumour and normal settings to enable sensitive somatic mutation detection from tumour tissues. We also attempted somatic DBS detection, which occurs in ~100 fold less frequently than SBS, but like somatic SBS detection, true DBS signatures were outweighed by DBS artefact signatures. 

We might be able to use a similar approach to also detect single molecule somatic structural variations. 

During CCS sequencing, the kinetics of DNA polymerase during DNA synthesis is recorded. How fast, slow and whether the DNA polymerase paused during DNA synthesis is recorded. DNA polymerase kinetics data can be used to determine the base modification such as 5mC. Dennis Lo and colleagues, for example, have used ctDNA and NIPT DNA CCS reads to detect 5mC from single molecules and to successfully use them as diagnostic markers \cite{Vong2019-bi, Tse2021-or}. Single molecule somatic mutation and 5mC together should provide greater sensitive with which tumours are classified, monitor their evolution and their potential trajectory under selection pressure. 

HMW DNA input requirements for PacBio CCS reads limit the use of CCS sequencing for NIPT and ctDNA based genetic diagnosis (discussed in Chapter 5). HMW DNA input requirements are, however, expected to decrease with library preparation optimisation and like how DNA input requirements for Illumina sequencing has decreased. 

Darwin Tree of Life project has sequenced and assembled high quality reference genomes using CCS and Hi-C reads, providing us with the opportunity to detect somatic mutations from other non-human samples, for the first time (discussed in Chapter 3). The somatic mutation rate and mutational signatures are unknown across these species. The study of somatic mutations across species allows us to tackle/attack the question posed by Peto's paradox: why doesn't species with greater number of cells don't have higher incidence of cancer?

We take advantage of the CCS base accuracy to detect gene conversions and crossovers in sperm samples and granulocytes from Bloom syndrome patients (discussed in Chapter 4). In addition,

PacBio has released new sequencing instrument Revio that increases the CCS read throughput 3 times with increase in read length and 3-fold increase in the number of ZMW, enabling the instrument to generate 30-fold sequence coverage genome at \$1000. This should drive adoption and increase the number of human genomes sequenced with the PacBio instrument. Researchers will typically use CCS reads for de novo assembly or for germline structural variation detection, but collection of CCS reads from public databases will enable the investigation of environmental mutagenesis across different populations across the globe and study the influence of germline mutation to somatic mutation generation and the combination of germline mutation and exogenous mutagen in generating new somatic mutagenesis. 

The introduction of himut allows researchers to detect 5mC, germline SNP, indel and structural variation detection and somatic mutation detection from a single SMRTcell on the Revio instrument. The breadth and depth of sequence and epigenetic information provided by CCS reads compared to Illumina sequencing for a single run of sequencing at a single molecule level should enable better diagnosis and study of samples. 

Three Matrix = Mutational signature probability 

Mutational signature is itself an abstraction of the three steps of somatic mutation: DNA damage, incorrect DNA repair and fixation. The accuracy of the PacBIo CCS bases and the ability to detect 5mC might enable us to dissect/deabstract the SBS1 mutational signature. The spontaneous deamination of 5mC to thymine (C>T) at CpG site is detected and repaired by the MMR repair machinery. We know the mutation probability of the spontaneous deamination of 5mC biological process to generate somatic mutations at CpG contexts, but we are, however, unaware of the rate at which spontaneous deamination of 5mC happens in vivo and the rate at which the C>T substitution is repaired and unrepaired by the mismatch repair (MMR) machinery. Using the base accuracy and the ability to detect 5mC base modification, we should be able to determine the rates of in vivo 5mC, success probability of the MMR machinery and the rate at which the C>T substitutions are fixed in the genome. We can imagine a scenario where a specific region will have wild type reads with 5mC, but one of the reads will have a C>T substitution. The subreads that was used to construct the CCS read can be examined to see whether the deamination happened on one of the strands and whether the other strand has complementary GC bases with 5mC. We can use similar approaches in the future to examine the probability of mutagen to generate DNA damage, DNA repair fidelity and DNA fixation probabilities. 

The application for our method abounds as our method can act as a replacement for many of the laborious processes that provide single-cell resolution somatic mutation calls. Our method cannot provide single-cell resolution somatic mutation calls, but we can provide through time-series sequencing of the same sample, the monitoring of the same somatic mutation to study the population dynamics of the sample. In addition, our method can be used to screen for ongoing mutational processes in the sample cheaply without needed to perform laborious single-cell clone expansion and sequencing. 
 















