% ************************** Thesis Abstract *****************************
% Use `abstract' as an option in the document class to print only the titlepage and the abstract.
\begin{abstract}

Pacific Biosciencesâ€™ circular consensus sequencing uses an engineered DNA polymerase with high processivity to repeatedly sequence the forward and reverse strand of a circular template. Leveraging redundancies and complementary base pairing between the forward and reverse strand, a highly accurate circular consensus sequence (CCS) read with at least Q20 read accuracy is generated. 

In this PhD thesis, I propose a hypothesis based on the similarities between duplex and CCS library preparation protocols. I conjecture that a CCS read is as accurate or more accurate than a duplex read, which is often used for ultra-rare somatic mutation detection. Using cord blood granulocytes with few somatic mutations, I demonstrate that a subset of CCS bases has higher base accuracy than duplex bases. In addition, I empirically calculate the CCS base accuracy to range from Q60 to Q90 depending on the substitution error and the trinucleotide sequence context, which is sufficiently accurate to enable single molecule somatic mutation detection in human samples and potentially in non-human samples with an unknown somatic mutation rate. 

To enable the somatic mutation detection and analysis across the tree of life, I design and develop a tool called himut to detect and phase somatic mutations from bulk normal tissue using CCS reads. I select a set of samples with a single somatic mutational process to distinguish recently acquired somatic mutations from false positive substitutions arising from sequencing and systematic bioinformatics errors. In addition, I ascertain that inaccurate estimation of base quality scores during CCS generation as one of the primary causes of false positive substitution detection. 

The Darwin Tree of Life (DToL) project aspires to sequence and generate reference genomes for around 70,000 eukaryotic species in Great Britain and Ireland. To date, the DToL consortium has publicly released CCS reads and reference genomes for approximately 600 eukaryotic species. I use himut to detect somatic mutations across the tree of life from DToL  datasets and perform \textit{de novo} mutational signature extraction to identify new and conserved somatic mutational processes. One striking observation is the episodic emergence and establishment of somatic mutational processes. The conservation of C>T somatic mutations at CG dinucleotides, resulting from spontaneous deamination of 5-methylcytosine to thymine, in animal, fungi and plant kingdom is one such example. I believe that methods described in this PhD thesis will facilitate the discovery and analysis of somatic mutational processes and enhance our understanding of natural selection. 

\end{abstract}
