% ************************** Thesis Abstract *****************************
% Use `abstract' as an option in the document class to print only the titlepage and the abstract.
\begin{abstract}

Pacific Biosciences' circular consensus sequencing utilises an engineered DNA polymerase with high processivity to sequence the forward and reverse strands of a circular template repeatedly. By leveraging redundancies and complementary base pairing between the two strands, a consensus sequence algorithm generates a highly accurate circular consensus sequence (CCS) read with a minimum read accuracy of Q20.

In this PhD thesis, I propose a hypothesis based on the similarities between duplex and CCS library preparation protocols. I conjecture that a CCS read is equally or more accurate than a duplex read, commonly employed for ultra-rare somatic mutation detection. Using cord blood granulocytes with few somatic mutations, I demonstrate that a subset of CCS bases exhibits higher accuracy compared to duplex bases. Furthermore, I empirically calculate the CCS base accuracy to range from Q60 to Q90, depending on the substitution error and trinucleotide sequence context. I show that a subset of CCS bases have sufficient base accuracy to enable single molecule somatic mutation detection in human samples and potentially in non-human samples with an unknown somatic mutation rate. 

To enable somatic mutation detection and analysis across the Tree of Life, I design and develop a tool called himut for detecting and phasing somatic mutations in bulk normal tissue using CCS reads. I select a set of samples with a single somatic mutational process to differentiate recently acquired somatic mutations from false positive substitutions caused by sequencing and systematic bioinformatics errors. Additionally, I identify inaccurate estimation of base quality scores during CCS generation as one of the main contributors to false positive substitution detection.

The Darwin Tree of Life (DToL) project aims to sequence and generate reference genomes for approximately 70,000 eukaryotic species in Great Britain and Ireland. As of now, the DToL consortium has publicly released reference genomes for around 600 eukaryotic species. I utilize himut to detect somatic mutations across different eukaryotic lineages that have been sequenced and assembled through the DToL project. Through \textit{de novo} mutational signature extraction, I also identify new and conserved somatic mutational processes across the Tree of Life. One striking observation is the episodic emergence and establishment of somatic mutational processes. An example of this is the conservation of C>T somatic mutations at CG dinucleotides, which result from the spontaneous deamination of 5-methylcytosine to thymine, in Animal, Fungi and Plant Kingdoms.
 
I believe that the methods described in this PhD thesis will facilitate the discovery and analysis of somatic mutational processes in normal tissue and across the Tree of Life
  

\end{abstract}
