%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Third Chapter **********************************
%*******************************************************************************
\chapter{Germline and somatic mutational processes across the tree of life}

% **************************** Define Graphics Path **************************
\ifpdf
    \graphicspath{{Chapter3/Figs/Raster/}{Chapter3/Figs/PDF/}{Chapter3/Figs/}}
\else
    \graphicspath{{Chapter3/Figs/Vector/}{Chapter3/Figs/}}
\fi


\section{Introduction}


\section{Introduction}

Somatic mutations can occur in cells at all stages of life and in all tissues. 

\subsection{The Darwin Tree of Life Project}

\subsection{CCS sequencing and \textit{de novo} assembly}

\section{Results}

\subsection{Somatic mutation detection}

\subsubsection{Phorcus lineatus somatic mutation rate}



\subsection{Germline and somatic mutational processes}

\section{Conclusion}



\section{Materials and Methods}

\subsection{CCS library preparation, sequencing and \textit{de novo} assembly}

To date, the DToL consortium has collected, prepared, and sequenced approximately $\sim$3000 eukaryotic samples in Great Britain and Ireland. In addition, reference genomes for around 600 eukaryotic species have been assembled and made available to the public, which is accompanied by a genome note that details the process from sample acquisition to chromosome-length scaffold construction. 

The DToL project initially used a combination of sequencing (CLR, CCS and linked reads) and scaffolding (e.g. Hi-C reads and BioNano genome maps) technologies to generate chromosome-length scaffolds. The DToL project currently uses HiFiAdapterFilter \cite{} to remove CCS reads with adapter sequences, either hifiasm \cite{} or hicanu \cite{} for \text{de novo} assembly of contigs from CCS reads, purgedups \cite{} to remove haplotype duplication, arrow \cite{} for contig polishing and SALSA \cite{} to order and orient contigs into chromosome-length scaffolds with Hi-C reads. If both the parent and child was sequenced, trio-canu was used to generate haplotype phased contigs. The chromosome-length scaffolds are, thereafter, manually curated using a Hi-C contact matrix to identify and correct misassemblies and to perform additional scaffolding where appropriate. If transcriptome data was available through either RNA or isoform sequencing, gene annotation was also performed in collaboration with the EMBL-EBI eukaryotic annotation team. The specific method and algorithm described here is subject to change with updates to the sequencing method, \textit{de novo} assembly and scaffolding algorithm. 

\subsection{\textit{Phorcus lineatus} somatic mutation rate measurement}

To calculate the somatic mutation rate of \textit{P. lineatus} (thick top shell), samples of different ages (3, 5 and 15) were collected from Plymouth, UK. Collaborators at the Marine Biological Association (MBA) determined the age of the samples from growth marks on the shells of samples. As recommended, a bench-mounted vice was first used to crush the shell and to carefully separate the sample from the shell (personal communication with Robert Mrowicki at the MBA). In addition, disposable scalpels were used during the dissection to prevent cross-contamination between the samples. HMW DNA was subsequently extracted from the foot muscle using the Circulomics Nanobind Tissue Big DNA Kit (SKU 102-302-100). CCS libraries were prepared following the low-input CCS library preparation protocol () and BluePippin system () was used to size select CCS libraries prior to sequencing.

CCS BQ score is a function of the number of supporting subreads and the concordance between the CCS base and subread bases. The DNAP processivity and CCS read length determine the number of full-length subreads per CCS read, which in turn influences the number of Q93 CCS bases from which potential somatic mutations can be identified. To account for the differences in the number of subreads per CCS read for each ZMW, the raw subreads BAM file was parsed using a custom script to select 10 full-length subreads per ZMW. The script calculates the median subread length for each ZMW and considers subreads between 0.9 times the median subread length and 1.1 times the median subread length as full-length subreads. The processed subreads BAM file was, thereafter, provided as an input to the pbccs algorithm to re-generate CCS reads. CCS reads were subsequently processed as described in chapter 2 and below. Except for the \textit{P. lineatus} somatic mutation rate measurement, CCS reads generated with default pbccs parameters for used for the rest of the analysis. 

\subsection{Germline and somatic mutation detection}

As detailed in chapter 2, CCS reads with adapter sequences were identified using HiFiAdapterFilt \cite{} and subsequently discarded. In addition, if ultra-low input CCS library preparation protocol was used for CCS generation and if this was documented, CCS reads were also excluded from downstream sequence analysis (this information, however, was not always available). CCS reads were, thereafter, aligned to the assembled reference genomes using minimap2 \cite{} and primary alignments were selected, sorted and merged into a single BAM file using samtools \cite{}. Germline mutations were called using deepvariant \cite{}. Somatic mutations were detected, and mutation burden was calculated using himut with default parameters. In addition, heterozygous SNPs were phased to construct haplotype blocks using himut, and haplotype phased CCS reads were subsequently used to call haplotype phased somatic mutations where applicable. Somatic mutation detection was again restricted to the autosomes of the reference genome. 

As CCS reads and reference genomes are derived from the same sample, homozygous germline mutations indicate assembly errors and analysis of germline mutations are restricted to heterozygous mutations for samples with a diploid genome.

The detection of somatic mutations across the tree of life followed a similar approach to the one described in chapter 2, but with minor modifications. When somatic mutations were called from DToL eukaryotic species, a VCF file containing germline mutations was supplied to himut to calculate heterozygosity ($\theta$) and the genotype prior $P(G)$. In addition, because a single sample was sequenced per species and as population-scale sequencing studies has not been performed for these species, PoN VCF file could not be generated and VCF file with common SNPs were not available for distinguishing false positive substitutions arising from systematic errors and gDNA contamination, respectively. However, given that CCS library and reference genome originate from the same sample, false positive substitutions arising from alignment errors should be minimal and CCS reads resulting from gDNA contamination should be excluded from the analysis based on their sequence identity.

\subsection{Mutational signature extraction and analysis}

As described in chapter 1, there are 6 substitution types (C>A, C>G, C>T, T>A, T>C, T>G) in the pyrimidine context and 16 trinucleotide sequence contexts for each substitution class, creating the canonical SBS96 classification system. Since the ancestral allele is known for somatic mutations, the SBS96 classification system is often used to categorise somatic substitutions. In contrast, because the ancestral allele is unknown for germline mutations, the SBS52 classification system is used for germline substitution classification. 

Here, I describe the SBS52 classification system and how the SBS96 classification system is transformed into the SBS52 classification system. The need for the SBS52 classification system arises from the fact that certain germline substitutions are indistinguishable from one another because the reference base cannot be assumed to be the ancestral allele; as the reference genome is sequenced and assembled from a randomly sampled individual, the haplotype containing the germline mutation could have also been the reference sequence. For instance, a C>A substitution in the AAA trinucleotide sequence context on the forward strand cannot be distinguished from a T>G substitution in the TTT trinucleotide sequence context on the reverse strand. Similarly, C>T substitutions cannot be differentiated from T>C substitutions. In addition, a C>G (T>A) substitution in a certain trinucleotide sequence context is interchangeable with another C>G (T>A) substitution in a different trinucleotide sequence context. Organised in Table \ref{tab:SBS52} is the complete transformation of the SBS96 classification system into the SBS52 classification system. 


\begingroup
\setlength{\LTleft}{-20cm plus -1fill} %% centering
\setlength{\LTright}{\LTleft}
\begin{longtable}{c|c|c}
\label{tab:SBS52} \\ \smallskip
SBS52 & \makecell{forward strand \\ (reference centred)} & \makecell{reverse strand \\ (read centred)}  \\ \hline
\ttfamily A[C>A]A & \ttfamily ACA>AAA = A[C>A]A & \ttfamily TTT>TGT = T[T>G]T \\ \hline
\ttfamily A[C>A]C & \ttfamily ACC>AAC = A[C>A]C & \ttfamily GTT>GGT = G[T>G]T \\ \hline
\ttfamily A[C>A]G & \ttfamily ACG>AAG = A[C>A]G & \ttfamily CTT>CGT = C[T>G]T \\ \hline
\ttfamily A[C>A]T & \ttfamily ACT>AAT = A[C>A]T & \ttfamily ATT>AGT = A[T>G]T \\ \hline
\ttfamily C[C>A]A & \ttfamily CCA>CAA = C[C>A]A & \ttfamily TTG>TGG = T[T>G]G \\ \hline
\ttfamily C[C>A]C & \ttfamily CCC>CAC = C[C>A]C & \ttfamily GTG>GGG = G[T>G]G \\ \hline
\ttfamily C[C>A]G & \ttfamily CCG>CAG = C[C>A]G & \ttfamily CTG>CGG = C[T>G]G \\ \hline
\ttfamily C[C>A]T & \ttfamily CCT>CAT = C[C>A]T & \ttfamily ATG>AGG = A[T>G]G \\ \hline
\ttfamily G[C>A]A & \ttfamily GCA>GAA = G[C>A]A & \ttfamily TTC>TGC = T[T>G]C \\ \hline
\ttfamily G[C>A]C & \ttfamily GCC>GAC = G[C>A]C & \ttfamily GTC>GGC = G[T>G]C \\ \hline
\ttfamily G[C>A]G & \ttfamily GCG>GAG = G[C>A]G & \ttfamily CTC>CGC = C[T>G]C \\ \hline
\ttfamily G[C>A]T & \ttfamily GCT>GAT = G[C>A]T & \ttfamily ATC>AGC = A[T>G]C \\ \hline
\ttfamily T[C>A]A & \ttfamily TCA>TAA = T[C>A]A & \ttfamily TTA>TGA = T[T>G]A \\ \hline
\ttfamily T[C>A]C & \ttfamily TCC>TAC = T[C>A]C & \ttfamily GTA>GGA = G[T>G]A \\ \hline
\ttfamily T[C>A]G & \ttfamily TCG>TAG = T[C>A]G & \ttfamily CTA>CGA = C[T>G]A \\ \hline
\ttfamily T[C>A]T & \ttfamily TCT>TAT = T[C>A]T & \ttfamily ATA>AGA = A[T>G]A \\ \hline
\ttfamily A[C>T]A & \ttfamily ACA>ATA = A[C>T]A & \ttfamily TAT>TGT = A[T>C]A \\ \hline
\ttfamily A[C>T]C & \ttfamily ACC>ATC = A[C>T]C & \ttfamily GAT>GGT = A[T>C]C \\ \hline
\ttfamily A[C>T]G & \ttfamily ACG>ATG = A[C>T]G & \ttfamily CAT>CGT = A[T>C]G \\ \hline
\ttfamily A[C>T]T & \ttfamily ACT>ATT = A[C>T]T & \ttfamily AAT>AGT = A[T>C]T \\ \hline
\ttfamily C[C>T]A & \ttfamily CCA>CTA = C[C>T]A & \ttfamily TAG>TGG = C[T>C]A \\ \hline
\ttfamily C[C>T]C & \ttfamily CCC>CTC = C[C>T]C & \ttfamily GAG>GGG = C[T>C]C \\ \hline
\ttfamily C[C>T]G & \ttfamily CCG>CTG = C[C>T]G & \ttfamily CAG>CGG = C[T>C]G \\ \hline
\ttfamily C[C>T]T & \ttfamily CCT>CTT = C[C>T]T & \ttfamily AAG>AGG = C[T>C]T \\ \hline
\ttfamily G[C>T]A & \ttfamily GCA>GTA = G[C>T]A & \ttfamily TAC>TGC = G[T>C]A \\ \hline
\ttfamily G[C>T]C & \ttfamily GCC>GTC = G[C>T]C & \ttfamily GAC>GGC = G[T>C]C \\ \hline
\ttfamily G[C>T]G & \ttfamily GCG>GTG = G[C>T]G & \ttfamily CAC>CGC = G[T>C]G \\ \hline
\ttfamily G[C>T]T & \ttfamily GCT>GTT = G[C>T]T & \ttfamily AAC>AGC = G[T>C]T \\ \hline
\ttfamily T[C>T]A & \ttfamily TCA>TTA = T[C>T]A & \ttfamily TAA>TGA = T[T>C]A \\ \hline
\ttfamily T[C>T]C & \ttfamily TCC>TTC = T[C>T]C & \ttfamily GAA>GGA = T[T>C]C \\ \hline
\ttfamily T[C>T]G & \ttfamily TCG>TTG = T[C>T]G & \ttfamily CAA>CGA = T[T>C]G \\ \hline
\ttfamily T[C>T]T & \ttfamily TCT>TTT = T[C>T]T & \ttfamily AAA>AGA = T[T>C]T \\ \hline
\ttfamily A[C>G]A & \ttfamily ACA>AGA = A[C>G]A & \ttfamily TCT>TGT = T[C>G]T \\ \hline
\ttfamily A[C>G]C & \ttfamily ACC>AGC = A[C>G]C & \ttfamily GCT>GGT = G[C>G]T \\ \hline
\ttfamily A[C>G]G & \ttfamily ACG>AGG = A[C>G]G & \ttfamily CCT>CGT = C[C>G]T \\ \hline
\ttfamily A[C>G]T & \ttfamily ACT>AGT = A[C>G]T & \ttfamily ACT>AGT = A[C>G]T \\ \hline
\ttfamily C[C>G]A & \ttfamily CCA>CGA = C[C>G]A & \ttfamily TCG>TGG = T[C>G]G \\ \hline
\ttfamily C[C>G]C & \ttfamily CCC>CGC = C[C>G]C & \ttfamily GCG>GGG = G[C>G]G \\ \hline
\ttfamily C[C>G]G & \ttfamily CCG>CGG = C[C>G]G & \ttfamily CCG>CGG = C[C>G]G \\ \hline
\ttfamily G[C>G]A & \ttfamily GCA>GGA = G[C>G]A & \ttfamily TCC>TGC = T[C>G]C \\ \hline
\ttfamily G[C>G]C & \ttfamily GCC>GGC = G[C>G]C & \ttfamily GCC>GGC = G[C>G]C \\ \hline
\ttfamily T[C>G]A & \ttfamily TCA>TGA = T[C>G]A & \ttfamily TCA>TGA = T[C>G]A \\ \hline
\ttfamily A[T>A]A & \ttfamily ATA>AAA = A[T>A]A & \ttfamily TTT>TAT = T[T>A]T \\ \hline
\ttfamily A[T>A]C & \ttfamily ATC>AAC = A[T>A]C & \ttfamily GTT>GAT = G[T>A]T \\ \hline
\ttfamily A[T>A]G & \ttfamily ATG>AAG = A[T>A]G & \ttfamily CTT>CAT = C[T>A]T \\ \hline
\ttfamily A[T>A]T & \ttfamily ATT>AAT = A[T>A]T & \ttfamily ATT>AAT = A[T>A]T \\ \hline
\ttfamily C[T>A]A & \ttfamily CTA>CAA = C[T>A]A & \ttfamily TTG>TAG = T[T>A]G \\ \hline
\ttfamily C[T>A]C & \ttfamily CTC>CAC = C[T>A]C & \ttfamily GTG>GAG = G[T>A]G \\ \hline
\ttfamily C[T>A]G & \ttfamily CTG>CAG = C[T>A]G & \ttfamily CTG>CAG = C[T>A]G \\ \hline
\ttfamily G[T>A]A & \ttfamily GTA>GAA = G[T>A]A & \ttfamily TTC>TAC = T[T>A]C \\ \hline
\ttfamily G[T>A]C & \ttfamily GTC>GAC = G[T>A]C & \ttfamily GTC>GAC = G[T>A]C \\ \hline
\ttfamily T[T>A]A & \ttfamily TTA>TAA = T[T>A]A & \ttfamily TTA>TAA = T[T>A]A \\ \hline
\caption{SBS52 classification and corresponding SBS96 classification}
\end{longtable}
\endgroup

After categorising germline and somatic substitutions based on the SBS52 and SBS96 classification system, SBS52 and SBS96 counts were processed as described below for \textit{de novo} mutational signature extraction using HDP \cite{}

\subsubsection{SBS96 mutational signature extraction}

As detailed in chapter 2, SBS96 counts were normalised according to the number of callable CCS bases, callable reference bases and the trinucleotide distribution in the autosomes of the reference genome. Somatic SBS96 counts in each species is a linear combination of true positive substitutions from somatic mutational processes and false positive substitutions from library, sequencing, and software errors. 

The number of false positive substitutions for each SBS96 classification, however, can be estimated from the substitution and trinucleotide sequence context dependent CCS error rate, which was determined in chapter 2, and the number of CCS trinucleotides from which somatic mutations could have been potentially called. These estimates can then be subtracted to obtain SBS96 counts where true positive substitution counts is better presented. If the same CCS library preparation protocol was not used for the DToL and the cord blood granulocyte sample, the estimation and subtraction of false positive substitutions from each SBS96 category may not be as effective in improving the signal-to-noise ratio. If a different protocol, for example, was used to extract HMW DNA and prepare a CCS library, false positive substitutions could be generated from an uncharacterized error process distinct from that identified in chapter 2.

After speciation, ongoing somatic mutational process(es) in germline stem cells depletes the trinucleotide sequence context it acts upon and shapes the trinucleotide distribution. To account for differences in trinucleotide distribution in each species, SBS96 counts are further normalised such that each trinucleotide sequence context equally contributes to the total SBS96 count (eq \ref{}). This normalisation further increases the signal-to-noise ratio of somatic mutational processes, particularly those with a higher somatic mutation rate and that are shared between the somatic and germline cells. 

Before \textit{de novo} mutational signature extraction, normalised SBS96 counts from each species are organised into a single matrix, samples with less than 100 somatic mutations are removed from the matrix and the total somatic mutation count for each species is normalised to the median somatic mutation count. After mutational signature extraction, each mutational signature was inspected for the following qualities to distinguish mutational signatures arising from ongoing somatic mutational processes in the sample or from library and sequencing errors upstream of sequence analysis. 

\begin{enumerate}
\item The mutational signature is similar to those found in the COSMIC mutational signature database.
\item The mutational signature is present in another species in the same phyla 
\item The mutational signature has biological replicates (e.g. idPlaAlba and xgPhoLine).
\item The mutational signature has transcriptional-strand bias
\item The mutational signature is similar to the germline mutational spectrum.
\item The attribution of the mutational signature to the total mutation burden in the sample 
\item The number of somatic mutations associated with the mutational signature is not a multiple of the number of germline mutations. 
\end{enumerate}

\subsubsection{Independent biological replication of mutational signatures}

To confirm that identified mutational signatures are the result of a biological process and not stochastic errors,

\subsubsection{Mutational signatures with transcriptional-strand bias}



\subsubsection{SBS52 mutational signature extraction}

To \textit{de novo} extract mutational signatures from germline mutations, germline and normalised somatic SBS52 counts from 518 eukaryotic species were organised into a single matrix and each SBS52 count was further normalised such that each trinucleotide sequence context contributes equally to the SBS52 count (eq. \ref{}). In contrast to the SBS96 classification system, the SBS52 classification system has 26 trinucleotide sequence contexts where the middle base is a pyrimidine base. 

Through mutational signature extraction from normalised germline and somatic SBS52 counts and downstream mutational signature analysis, ancestral alleles of germline mutations were recovered and the contribution of somatic mutational processes to the germline mutational spectrum was also measured. 

\subsection{Timing the emergence of somatic mutational processes}

The phylogenetic relationship between 518 species and the time of speciation was inferred from phylogenetic tree available at http://www.timetree.org and relevant information from academic literature. The birth of new species with new somatic mutational processes was used to time the emergence of new somatic mutational processes. The time at which the new somatic mutational process is estimated to have emerged will have to be updated with the ongoing efforts from the DToL consortium.  


