%!TEX root = ../thesis.tex
%*******************************************************************************
%*********************************** First Chapter *****************************
%*******************************************************************************

\chapter{Introduction}  %Title of the First Chapter

\ifpdf
    \graphicspath{{Chapter1/Figs/Raster/}{Chapter1/Figs/PDF/}{Chapter1/Figs/}}
\else
    \graphicspath{{Chapter1/Figs/Vector/}{Chapter1/Figs/}}
\fi


%********************************** %First Section  **************************************

\section{Somatic mutations}


\subsection{Somatic mutation detection}

\subsection{Somatic mutations in normal tissues}

\subsection{Somatic mutational processes}

\subsection{Mutational signature analysis}



Somatic mutations can occur in cells at all stages of life and in all tissues. The biochemical manifestation of a somatic mutation requires three distinct stages: DNA damage or modification from either endogenous or exogenous sources, mutation resulting from incorrect DNA damage repair and unrepaired DNA damage, and the persistence of the mutation in the genome of the cell and its descendants \cite{Stratton2009-of}. Most somatic mutations are benign, but some confer a proliferative advantage and are referred to as driver mutations. The advent of next-generation sequencing and the continued decline in sequencing costs have enabled us to sequence thousands of cancer genomes at scale and subsequent downstream sequence analysis has allowed us to discover tissue-specific driver mutations \cite{Martinez-Jimenez2020-kn}, identify biological processes that generate these mutations \cite{Alexandrov2013-kg}, to use somatic mutations as timestamps and biological barcodes to lineage trace development \cite{Behjati2014-gb}, to discover complex structural rearrangements such as chromothripsis \cite{Stephens2011-gj} that fundamentally changed the conventional view of tumorigenesis as the gradual process of the accumulation of somatic mutations \cite{Doll1954-of, Knudson1971-fg} and to better understand the relationship between abnormal embryonic development and paediatric tumour formation \cite{Marshall2014-ec}. International efforts such as the Cancer Genome Atlas (TCGA) program \cite{Weinstein2013-ko} and the Pan-Cancer Analysis of Whole Genomes (PCAWG) consortium \cite{ICGCTCGA_Pan-Cancer_Analysis_of_Whole_Genomes_Consortium2020-ts} have also measured and analysed genetic, epigenetic, transcriptomic and proteomic aberrations in thousands of tumour genomes to understand how these aberrations contribute to the hallmarks of cancer \cite{Hanahan2000-dp, Hanahan2011-zr}. 

Cancer is often described as the disease of the genome. Somatic mutation detection, hence, is often the first step towards characterising the cancer genome and these somatic mutations have been catalogued and analysed to determine their contribution to tumorigenesis. Multiple mutational processes simultaneously act on the genome at any given time and contribute to the accumulation of somatic mutations over an individual’s lifetime. To determine the mutational sources from a set of samples, mutational signature analysis is performed to either \textit{de novo} extract new mutational signatures or to assign the contribution of known mutational signatures to the mutation burden \cite{Alexandrov2013-fq}; a mutational signature is a mathematical abstraction of the likelihood that a particular biological process will produce a somatic mutation in a specific sequence context. During mutational signature analysis, somatic mutations are classified according to the event, the size of the event and the sequence context. Single base substitutions (SBS), for example, can be classified using the SBS96 classification system, which categorises SBS according to the six types of substitutions in the pyrimidine context (C>A, C>G, C>T, T>A, T>C and T>G) and the 16 possible trinucleotide sequence contexts derived from the 4 possible bases upstream and downstream of the substitution. SBS can be further subclassified based on their pentanucleotide sequence context (SBS1536 classification) and whether the SBS is located on the intergenic DNA, transcribed or untranscribed strand of the gene (SBS288 classification). Double base substitution, indel and structural variation classification system also exist for mutational signature analysis, but they are not the subject of interest in this chapter \cite{Alexandrov2013-fq, Li2020-vw, Steele2022-mn}. 

The PCAWG consortium has discovered 67 single-base-substitution (SBS) mutational signatures \cite{Alexandrov2020-ys}. To date, the biological aetiology for 49 SBS mutational signatures has been determined (Table X). The discovery of new somatic mutational signatures is an ongoing process where the number and the aetiology of mutational signatures is constantly updated and refined with increase in the number of sequenced genomes. Genomics England and collaborators, for example, have leveraged 100,000 cancer genomes from around 85,000 patients to detect mutational signatures associated with rare and sporadic somatic mutagenesis \cite{Degasperi2022-qe}. In addition, somatic mutations resulting from chemotherapeutic agents is another active area of research \cite{Pich2019-ja, Aitken2020-sa}. Clinical sequencing of matched tumour and normal genomes is now routinely performed in the developed countries to help cancer patient treatment, fulfilling one of the many promises of the human genome project..

Somatic mutation detection, however, is not a solved problem. Somatic mutation callers, for example, employ different strategies and exhibit varying specificities and sensitivities. Consensus somatic mutation call from multiple different detection algorithms, hence, is often used for downstream analysis \cite{Bailey2020-ou}. The base accuracy and read length of Illumina reads, most importantly, is the common technical factor that limits the resolution at which the somatic mutations can be detected. MuTect, for example, cannot differentiate Illumina sequencing errors from low variant allele fraction (VAF) somatic mutations as a typical Illumina base call has a 0.01-1\% error rate \cite{Cibulskis2013-gw}. Library errors, introduced upstream of sequencing, are also often misclassified as somatic mutations \cite{Costello2013-cz, Chen2017-ba, Abascal2021-pk}. Newly acquired somatic mutations, therefore, are indistinguishable from background noise using conventional methods and required breakthroughs in sample and library preparation (Figure \ref{}). The detection of these somatic mutations, however, are critical for early detection of cancer, monitoring of tumour evolution during patient treatment and to enhance our understanding of the transformation of normal cells to neoplastic cells. 

The repeat content of the genome is another hurdle for accurate somatic mutation detection. Repetitive sequences (e.g., tandem repeat expansions, retrotransposons, segmental duplications, telomeric repeats and centromeric alpha-satellite) account for approximately 50\% of the human genome \cite{Lander2001-du}. If the repeat length is greater than the read length, read alignment software cannot determine the location of the read with respect to the reference genome as the read could have originated from any copies of the repetitive sequence \cite{Li2008-dt}. The accurate placement of reads, hence, requires repetitive sequences to be flanked with unique sequences not present elsewhere in the reference genome. Consequently, the reference genome is divided into callable region and non-callable regions based on mappability of Illumina short reads and variant calling is often restricted to the callable regions of the genome \cite{1000_Genomes_Project_Consortium2012-rj}. Clinically relevant genes in non-callable regions, hence, are often excluded from analysis \cite{Wagner2022-ph}.

The completeness and contiguity of the reference genome is another often ignored, but important factor, for somatic mutation detection. The human reference genome constructed from physical mapping and clone-by-clone sequencing and assembly of overlapping BAC clones is undoubtedly the best mammalian reference genome \cite{Lander2001-du}, but the human reference genome is still incomplete. The human reference genome, for example, still has missing sequences, unplaced scaffolds and unlocalised scaffolds without a reference coordinate, and misassemblies such as incorrect sequence collapse and expansion. Furthermore, approximately 70\% of the human reference genome is derived from genomic DNA of an anonymous individual of African-European ancestry \cite{Osoegawa2001-np}. The current linear sequence of the human reference genome, therefore, may not accurately reflect the genomic diversity present in other populations and alternative graph-based representations might better incorporate genomic diversity \cite{Garrison2018-ae}. The Genome Reference Consortium (GRC) has released GRCh38 build with alternative loci to address some of these issues \cite{Schneider2017-yo}. The recent completion of telomere-to-telomere CHM13 (T2T-CHM13) haploid genome using a combination of sequencing and mapping technologies has been a major milestone for genomics research \cite{Nurk2022-dv}. T2T-CHM13 genome, as expected, improve the accuracy and precision of both read alignment and variant calling \cite{Aganezov2022-dv}. 

% Table of current somatic mutation callers, their sensitivity and specificity, and their approaches \cite{}.

\subsection{Challenges in somatic mutation detection}

Cancer is often described as the disease of the genome. Somatic mutation detection, hence, is often the first step towards characterising the cancer genome and 

these somatic mutations have been catalogued and analysed to determine their contribution to tumorigenesis. 

Somatic mutation detection, however, is not a solved problem. Somatic mutation callers, for example, employ different strategies and exhibit varying specificities and sensitivities. Consensus somatic mutation call from multiple different detection algorithms, hence, is often used for downstream analysis \cite{Bailey2020-ou}. The base accuracy and read length of Illumina reads, most importantly, is the common technical factor that limits the resolution at which the somatic mutations can be detected. MuTect, for example, cannot differentiate Illumina sequencing errors from low variant allele fraction (VAF) somatic mutations as a typical Illumina base call has a 0.01-1\% error rate \cite{Cibulskis2013-gw}. Library errors, introduced upstream of sequencing, are also often misclassified as somatic mutations \cite{Costello2013-cz, Chen2017-ba, Abascal2021-pk}. Newly acquired somatic mutations, therefore, are indistinguishable from background noise using conventional methods and required breakthroughs in sample and library preparation (Figure \ref{}). The detection of these somatic mutations, however, are critical for early detection of cancer, monitoring of tumour evolution during patient treatment and to enhance our understanding of the transformation of normal cells to neoplastic cells. 

\subsection{Single molecule somatic mutation detection}

Illumina’s technical limitations have limited somatic mutation detection to clonal or sub-clonal mutations. Two approaches have been developed to address these challenges: 1) to increase the copy number of the mutant DNA above the limit of detection threshold and 2) to increase the base accuracy of the Illumina reads through upstream changes in the library preparation protocol. Single-cell whole-genome amplification \cite{Lodato2018-hh}, single-cell clone expansion \cite{Lee-Six2018-qe} and laser-capture microdissection (LCM) \cite{Ellis2021-it} and sequencing adopts the former approach. Rolling circle amplification \cite{Lizardi1998-qh, Dahl2004-tm} and duplex sequencing methods \cite{Schmitt2012-yr, Abascal2021-pk, Hoang2016-jx} adopt the latter approach where a highly accurate consensus sequence is created from multiple copies of a single molecule. 

Single-cell clone expansion and LCM sequencing are recognized as the gold-standard methods for somatic mutation detection in single-cells or clonal tissues, respectively. These methods have enabled the study of embryogenesis, somatic mutation rate, mutational processes, clonal structure, driver mutation landscape and earliest transformation of normal cells to neoplastic cells across a range of normal tissues, including adrenal gland, blood, bladder, bronchus, cardiac muscle, colon, endometrium, oesophagus, pancreas, placenta, prostate, skin, smooth muscle, testis, thyroid, ureter, visceral fat \cite{Lee-Six2018-qe, Martincorena2015-gu, Ju2017-vw, Martincorena2018-av, Brunner2019-xg, Lee-Six2019-vt, Yoshida2020-yr, Olafsson2020-vi, Moore2020-pi, Lawson2020-em, MSpencer_Chapman2021-cq, Coorens2021-ct, Robinson2021-te, Grossmann2021-gd, Moore2021-dl, Park2021-fx, Ng2021-jd}. Duplex sequencing, however, is the most scalable option for ultra-rare somatic mutation detection and is the preferred method for circulating tumour DNA (ctDNA) based clinical applications \cite{Newman2016-cy}.

\section{Single molecule sequencing}

Single molecule sequencing technologies from Oxford Nanopore Technologies (ONT) and Pacific Biosciences (PacBio) are spearheading the next decade of the genomics revolution. These upcoming technologies promise a new era of genomics where: 1) lower input material is required for library preparation and sequencing, 2) library preparation is location-agnostic and does not require skilled technicians, 3) sequencing takes hours and not days, 4) higher base accuracy, 5) longer read length (10kb – 100kb), 6) simultaneous detection of genetic variations and base modifications and 7) nucleotide-resolution identification of structural variations where event size is $\geq$50bp. Despite these promising capabilities, the higher error rate and marginally longer read length of first generation of ONT reads and PacBio continuous long reads (CLR) limited their adoption. Illumina is still the primary sequencing method in most labs as per base sequencing cost is still cheaper with the Illumina platform. After decades of development, ONT and PacBio have introduced new sequencing instruments and library preparation that exceeds the capabilities of Illumina platform in read length and accuracy, enabling researchers to access the inaccessible regions of the genome and to explore biological phenomena that could not be explored before.

\subsection{Nanopore sequencing}

Cells use membrane proteins to move ions and molecules, critical to the maintenance of cellular function, across the permeable plasma membrane through passive and active transport \cite{}. David Deamer and George Church independently hypothesised 197X that a single strand of DNA molecule could be passed through a protein pore if voltage is applied through the membrane holding protein pore cite{}. If electrostatic potential is present across the protein. The disruption of the passage of ionic currents by the passage of the DNA molecule through the pore can be recorded and can be interpreted as a specific nucleotide base. Nanopore based sequencing methods promised 1) minimal library preparation, 2) ultra-fast native DNA and RNA sequencing and 3) unlimited read length 

Today, Oxford Nanopore Technologies (ONT) has fulfilled many of these promises. To fulfil these promises, Deamer and colleagues had to demonstrate the potential of the Nanopore sequencing method through successive demonstrations and improvements of the technology that first shows that passage of the DNA through a pore and disruption of the ionic current is a detectable event \cite{}, and that a single nucleotide difference can be detected from a background of homopolymer sequence \cite{}. In addition, the first generation of pores based on $alpha$ had a pore that was too long such that 10-15 nucleotides will be interpreted as a single signal and hence, a pore that had a similar aperture, but shorter pore was required to improve the signal-to-noise ratio. MytA protein, hence, thereafter, was used for nanopore sequencing to improve the signal-to-noise ratio. To improve their base accuracy, ONT introduced 2D reads where both forward and reverser strand of the double-stranded DNA with a hairpin adapter could be sequenced through the nanopore \cite{}. ONT, however, long no supports 2D reads as a result of legal dispute with PacBio \cite{}.

ONT licensed these patents to commercialise the technology in 2005 and the most recent ONT reads are reported to have Q20 read accuracy \cite{}. To date, ONT reads have been successfully used to identify and characterise complex pathogenic mutations\cite{},  accelerate clinical diagnosis \cite{}, and to help the assembly of the complex regions in the human reference genome \cite{}. It could be said that ONT sequencing has fulfilled all of it promises and more.   


\subsection{Pacific Biosciences Single-Molecule Real-Time sequencing}

PacBio was founded in 2004 with aspirations to commercialise single molecule real time (SMRT) sequencing technology developed at Cornell University. The SMRT platform is the culmination of multiple technical innovations from a range of disciplines. The zero-mode-waveguide (ZMW), a nano-scaled hole fabricated in a metal film, for example, is at the heart of the SMRT platform. The ZMW acts as the sequencing unit and its unique properties help the SMRT platform achieve the high signal-to-noise ratio required to observe activity of individual DNA polymerases (DNAP)\cite{Levene2003-og}. 

The metal film with the ZMW is placed on top of a glass and DNAP is immobilised at the bottom glass surface through surface chemistry modifications that prevent the adsorption of DNAP to the metal side walls\cite{Korlach2008-aq, Eid2009-ol}. A topologically circular template, also known as a SMRTbell template, is created through the attachment of hairpin adapters to a double-stranded DNA molecule (Figure X). The successful loading of SMRTbell template into a ZMW follows a Poisson distribution and typically 30 to 50\% of the ZMWs are classified as productive ZMWs where a single DNAP successfully initiates and completes rolling circle amplification. SMRT sequencing initially used $\Phi$29 DNAP for its high processivity, minimal amplification bias and ability to perform strand displacement DNA synthesis \cite{Eid2009-ol}. In addition, $\Phi$29 DNAP was engineered through site-directed mutagenesis to use fluorophore-labelled deoxyribonucleoside triphosphate (dNTP) during DNA elongation \cite{Korlach2008-fv,Eid2009-ol}. 

Upon successful loading of SMRTbell templates, free nucleotides are released above the ZMW array and free nucleotides diffuse in and out of the ZMW. DNAP binds and incorporates the correct nucleotide into the growing DNA strand, and upon nucleotide incorporation, DNAP cleaves the fluorophore from the nucleotide such that the synthesised DNA molecule consists of native DNA molecules. DNAP continues DNA elongation until DNA replication is terminated. The length of the reaction time is dependent on DNAP processivity and the presence of bulky DNA damage on the template DNA that can lead to premature termination of replication\cite{}. Illumination from the laser below the glass surface excites the fluorophore and the emitted fluorescence is measured. Image processor leverages the temporal difference between diffusion of free nucleotides (which occurs in microseconds) and nucleotide incorporation (which occurs in milliseconds) to separate the background fluorescence from free nucleotides and fluorescence from nucleotide bound to DNAP. In addition, the size and shape of the ZMW prevents laser light from passing through the ZMW and limits the illumination to the bottom of the ZMW, which further increases the signal-to-noise ratio. As the four dNTPs are each labelled with a different fluorophore, each nucleotide can be identified from their unique fluorescence\cite{Eid2009-ol}. DNA base modification detection can also be achieved from analyzing DNAP kinetics, which consist of duration of fluorescence pulse, known as pulse width, and the duration between successive fluorescence pulses, referred to as interpulse duration \cite{Flusberg2010-ub}. To date, DNAP kinetics have been used to detect base modifications such as N6-methyladenine, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine \cite{Flusberg2010-ub} and DNA damage such as O6-methylguanine, 1-methyladenine, O4-methylthymine, 5-hydroxycytosine, 5hydroxyuracil, 5-hydroxymethyluyracil and thymine dimers \cite{Clark2011-jz}. 

SMRT platform capability was initially limited to continuous long read (CLR) generation with 10-15\% error rate \cite{Eid2009-ol} instead of circular consensus sequence (CCS) generation with 0.1-1\% error rate \cite{Wenger2019-pw}. This was because there is an inherent trade-off between read length and read accuracy while DNAP processivity is held as a constant. The earlier generations of DNAP had insufficient processivity to sequence both the forward and reverse strand of a SMRTbell template multiple times. In contrast, the more recent generations of DNAP have sufficient processivity to sequence the forward and reverse strand of long SMRTbell templates (>10kb) multiple times such that both long and accurate reads are produced \cite{Wenger2019-pw}. SMRT platform, hence, leveraged the improvements in DNAP processivity to first increase read length and subsequently improve read accuracy. 

The PacBio RS instrument with the first generation of polymerase and chemistry (P1-C1) produced continuous long reads (CLR) with an average read length of 1,500 bp with 10-15\% error rate \cite{}. In contrast, the most recent PacBio Revio instrument generates circular consensus sequence (CCS) reads with an average read length of 20,000 bp with 0.1-1\% error rate \cite{}. In addition, the PacBio RS instrument used the first generation of SMRTcell with 150,000 ZMWs \cite{} while the PacBio Revio instrument uses the latest SMRTcell with 25 million ZMWs, increasing the sequence throughput exponentially from 22 million bases to 90 billion bases per SMRTcell \cite{} (Figure \ref{}). Compared to Illumina sequencing, CLR sequencing had a higher error rate and cost per-base, with only marginal increases in read length. In addition, the shortage of bioinformatics algorithms to process CLR reads with high error rate also slowed market adoption. The PacBio Revio instrument, however, can generate 30-fold CCS sequence coverage of the human genome under \$1000. The sequence data from a single SMRTcell, therefore, can be used for not only \textit{de novo} assembly \cite{} but also haplotype-phased base modification\cite{}, SNP and indel, \cite{} and structural variation detection \cite{}, enabling the most comprehensive characterisation of both genetic and epigenetic variation from a single human individual. We also expect the sequence throughput per SMRTcell to increase exponentially in the foreseeable future with improvements in DNA processivity that increases CCS read length and advances in semiconductor fabrication technologies that doubles or triples the number of ZMWs per SMRTcell. 

\subsection{Long-read sequencing applications}

In the beginning, long reads from ONT and PacBio SMRT platform did not have a competitive advantage compared to short reads from Illumina platform; long reads were only marginally longer than short reads and their higher error rate made accurate germline mutation detection more challenging. Long-read sequencing, most importantly, could not compete with short-read sequencing on sequencing cost. 

\subsubsection{\textit{De novo} assembly}
A substantial increase in read length from ~1,500 bp to 10,000 bp with the introduction of XX chemistry for ONT and P5-C3 chemistry for PacBio Sequel I instrument reignited interest for new \textit{de novo} assembly algorithm development, full-length transcript sequencing and accessing the inaccessible regions of the genome.

Genomes are peppered with repetitive sequences. These repetitive sequences, for example, account for more than 50\% of the human genome\cite{Lander2001-du}. The unique placement of a read in an assembly graph, therefore, requires read length to be longer than the repeat length such that unique sequences not found elsewhere in the genome flank the repetitive sequence in the read. Gaps and collapsed regions in genome assemblies, hence, often result from regions of the genome where the repeat length is longer than read length. There are, however, not many repeats except for segmental duplications\cite{Bailey2002-xn}, higher order repeats (HOR) in centromeres\cite{Willard1985-bo} and palindromic sequences in sex chromosomes that are longer than ONT and CLR reads \cite{Skaletsky2003-sr}. 

A new generation of assembly algorithms based on de Brujin graph\cite{Lin2016-vl}, string graph\cite{Myers2005-ei, Chin2016-at} and OLC\cite{Koren2017-cq} were developed to leverage these long reads and enable end-to-end assembly of microbial genomes\cite{Bashir2012-cs, Chin2013-hp} and large mammalian genomes\cite{Chin2016-at, Koren2017-cq}. Complete hydatidiform mole (CHM) 1 BAC clones, for example, were selected for hierarchical shotgun sequencing to close existing gaps in the human reference genome \cite{Huddleston2014-rs}. At the time, contigs produced from these new assembly algorithms had unparalleled contiguity as measured by contig N50 \cite{}. In addition, misassemblies can be corrected, and contigs can be ordered and oriented into scaffolds using optical genome maps from Bionano Genomics \cite{Pendleton2015-ue}. Chromosome-length scaffold construction, more importantly, has become routine through Hi-C scaffolding\cite{Dudchenko2017-kb} and the ability to visualise\cite{Robinson2018-os} and manually inspect Hi-C contact matrix for assembly curation\cite{Dudchenko2018-yl}. Trio-sequencing\cite{Koren2018-wg} and single-cell strand sequencing data\cite{Porubsky2021-ct} have also been used to also construct haplotype-resolved assemblies. These chromosome-length scaffolds, most importantly, are often comparable or better than existing reference genomes in both contiguity and completeness \cite{Matthews2018-tv}. 

Ultra-long read library preparation from ONT and CCS library preparation from PacBio were two additional breakthroughs that transformed how \textit{de novo} assembly is performed today. Ultra-long reads (>100kb) have been particularly useful for closing gaps\cite{Jain2018-zh} and for full-length sequencing of overlapping BAC clones for assembly of human chromosome Y centromere\cite{Jain2018-mg}. Human centromeres are enriched with AT-rich 171 bp tandem repeats called $\alpha$-satellite DNA. Centromeric $\alpha$-satellite DNA organises into HOR structures that are several megabases in length. Despite their crucial role in cell division, the organisation and structure of human centromeres were inaccessible to interrogation until the introduction of ultra-long reads. It is worth mentioning that centromeres in the b37 and hg38 reference genome are recorded as sequences and therefore do not provide a true representation of the underlying sequence \cite{Miga2014-uv}.

CCS read length and accuracy have been leveraged to reduce computational complexity of all-to-all pairwise read alignments and shorten genome assembly time \cite{Chin_undated-ye} and to distinguish recently diverged haplotypes and repeat copies such as segmental duplications \cite{Nurk2020-gu, Cheng2021-ij}. CCS reads are, routinely, used to produce haplotype-resolved chromosome-arm length contigs. It is worth mentioning that assembly algorithms often assume that the sample in question has a haploid genome. This assumption results in haplotype collapsed assemblies where the assembled haplotype is not present in the population \cite{Schneider2017-yo}. The completion of telomere-to-telomere (T2T) CHM13 (T2T-CHM13) genome, including the short arms of five acrocentric chromosomes and centromeric satellite array, has been the culmination of years of effort to produce gapless and error-free assemblies \cite{Nurk2022-dv}. These advancements allow us to construct high-quality reference genomes for a fraction of what it used to cost to build the human reference genome. The number of new plant and animal assemblies has burgeoned thanks to these developments \cite{}. 

\subsubsection{Full-length transcript sequencing}

In contrast, to short-read sequencing that requires \textit{de novo} assembly of RNA reads to acquire full-length transcripts, long-read sequencing can be used to obtain full-length transcript without assembly. Long-read sequencing has been used to successfully identify new isoforms in tissues and novel gene fusions in cancers \cite{}. Single-cell isoform-sequencing has also been used to find new isoforms, to define the transcriptome atlas and to quantify the transcript in combination with single-cell RNA sequencing. In addition, these full-length transcripts have been successfully used for gene annotation of newly assembled genomes \cite{}.

\subsubsection{Germline and somatic mutation detection}


To date, ONT, CLR and CCS reads have been successfully used for germline SNP, small insertion and deletion\cite{} and structural variation detection \cite{}. The lower base accuracy and higher per base sequencing cost has limited the use of ONT and CLR reads for SNP and indel detection. The longer read length, however, enabled access to regions of the genome inaccessible with short reads and early success in identification of pathogenic mutations in undiagnosed patients with rare diseases \cite{}.

Structural variation detection with short reads relies on either changes in sequence coverage for copy number variation (CNV) detection and identification of discordant read pairs with aberrant distance and orientation for breakpoint, translocation and inversion detection \cite{Alkan2011-dv}. In contrast, long reads enable structural variation detection with nucleotide resolution through direct comparison of read and reference genome and is also more sensitive towards short tandem repeat (STR) expansions, short interspersed nuclear element (SINE) and long interspersed nuclear elements (LINE) insertion detection \cite{Chaisson2015-zz, Sedlazeck2018-oh, Denti2022-ux}. CHM1 CLR reads, for example, were also used to correct small misassembles in the reference genome and identify approximately 26,000 structural variations that were recalcitrant to detection using short reads \cite{Chaisson2015-zz}; the number of structural variations detected with long reads is at least double that detected with short reads. The number of structural variations is orders of magnitude smaller than the number of SNPs and indels, but structural variations alter greater number of bases and have a more pronounced impact on speciation and phenotype through gene regulation, duplication, translocation\cite{Weischenfeldt2013-tl} and conformational changes in three-dimensional genome configuration\cite{Spielmann2018-fm,}. In addition, complex structural rearrangements such as chromothripsis\cite{Stephens2011-gj, Korbel2013-to}, chromoplexy\cite{Baca2013-po} and templated insertions\cite{Yu2010-jr} are common oncogenic mechanisms. Repeat expansions and accompanied hypermethylation are common causes of neurological diseases\cite{Zhou2022-ci}. The severity of Parkinson’s disease, for example, is associated with repeat content and the size of the repeat expansion\cite{}. Single-molecule sequencing is the only reliable technology for repeat expansion detection. Low genetic diagnosis rate of approximately 30\% with short read sequencing and ability to detect haplotype phased genetic and epigenetic variations with single molecule sequencing has renewed interest to detect causal and putative pathogenic mutations in patients with rare genetic disease\cite{}. 

Despite the advantages that long-read sequencing technologies offers compared to short-read sequencing technologies for somatic structural rearrangement detection, the application of long-read sequencing technologies to somatic mutation detection has been limited to date. There has been a handful publications that interrogated somatic structural rearrangements in breast cancer cell lines with long reads \cite{}. Somatic mutation detection with long reads is at the stage where we are re-creating the capabilities provided by short-sequencing technology and is not at the stage where we are finding somatic mutations that cannot be detected with short-read sequencing technology.

Structural variation detection with short reads relies on either changes in sequence coverage for copy number variation (CNV) detection or identification of discordant read pairs with aberrant distance and orientation for breakpoint, translocation and inversion detection \cite{Alkan2011-dv}. In contrast, long reads enable structural variation detection with nucleotide resolution through direct comparison of read and reference genome and are also more sensitive towards short tandem repeat (STR) expansions, short interspersed nuclear element (SINE) and long interspersed nuclear elements (LINE) insertion detection \cite{Chaisson2015-zz, Sedlazeck2018-oh, Denti2022-ux}. CHM1 CLR reads, for example, were also used to correct small misassemblies in the reference genome and identify approximately 26,000 structural variations that were recalcitrant to detection using short reads \cite{Chaisson2015-zz}; the number of structural variations detected with long reads is at least double that detected with short reads. The number of structural variations is orders of magnitude smaller than the number of SNPs and indels, but structural variations alter a greater number of bases and have a more pronounced impact on speciation and phenotype through gene regulation, duplication, translocation\cite{Weischenfeldt2013-tl} and conformational changes in three-dimensional genome configuration\cite{Spielmann2018-fm,}. In addition, complex structural rearrangements such as chromothripsis\cite{Stephens2011-gj, Korbel2013-to}, chromoplexy\cite{Baca2013-po} and templated insertions\cite{Yu2010-jr} are common oncogenic mechanisms. Repeat expansions and accompanied hypermethylation are common causes of neurological diseases\cite{Zhou2022-ci}. The severity of Parkinson’s disease, for example, is associated with repeat content and the size of the repeat expansion\cite{}. Single-molecule sequencing is the only reliable technology for repeat expansion detection. Low genetic diagnosis rate of approximately 30\% with short read sequencing and ability to detect haplotype phased genetic and epigenetic variations with single molecule sequencing has renewed interest to detect causal and putative pathogenic mutations in patients with rare genetic disease\cite{}. 

\section{The Darwin Tree of Life Project}

We cannot understate the significance that we can study physics and chemistry anywhere in the universe, but we can only study biology on planet Earth. We search for signs of life elsewhere in the universe, but we have yet to succeed in this endeavour. We must, hence, assume what distinguishes the inanimate from animate can be only understood here on Earth. 

%7. We know very little about the somatic mutation landscape outside of humans (worth a paragraph on Alex's mammals paper though...)
%8. Why we might want to understand somatic mutational processes across the tree of life - 
%8a. Somatic mutations as a barometer of mutagenic exposures in the environment
%8b. Understanding adaptation to toxic niches
%8c. Comparative studies across species for ageing
%8d. As a route to understanding processes underlying germline evolution across the tree of life

\subsection{Origins of Life, Prebiotic Earth, RNA world, The emergence and evolution of life on Earth}
%
%Ribozyme
%Endosymbiosis
%Phagocytosis
%energy: mtDNA
%archaea
%eukaryotes: organelles
%prokaryotes
%oxidation and photosynthesis
%sex: meiotic recombination
%Oxidation
%
%We start of time, entropy has been increasing following the second law of thermodynamics and biological systems have emerged to reduce or maintain entropy using energy. Phospholipid permeable-membrane was the first spontaenous invention that separated order from disorder and allowed for the movement of molecules between the extracelular and intercellular environemtnt and for the emergence of primordial cell. It is uncertain whether the first cell had both the capacity to recplicate itself or whether had the capacity to catalyze chemical reaction first. In a prebiotic environment, amino acids can be created in a reducing environment if sufficient energy in the form of ioniznig radiation, ultra-violet light, is introduced into a gaseous atmosphere containing methane, ..., ... and ... [ref] and nucleotide bases are thought to be harder to spotaneously create in a prebiotic envrionment [ref]. Despite the uncertainity in how the first cell arised, the first prokaryotic organism is thought to have arise ~XX billion years ago and the first eukaryotic organism is thought to have arisen approximately 2 billion years ago [ref]. Once the first cell was created, selection pressure and natural selection acted upon these cells to create the first multicelullar organism and these muticellular organisms evolved to create multiple different species that is best adapated to the environment surrounding them. Mutations play a central role in creating new innovations that allows for individual species to better adapt to the environment and to produce progenitors that inherit the mutations. 

The advent of high-throughput long-read sequencing\cite{} and genome mapping technologies\cite{}, improvements in base accuracy of long reads \cite{Wenger2019-pw} and development of algorithms that take advantage of the longer read length and long-range genomic interactions \cite{Dudchenko2017-kb} have brought new enthusiasm to sequence and assemble high-quality reference genomes\cite {}. 

The Darwin Tree of Life (DToL) project is an ambitious project that aspires to construct chromosome-length scaffolds for 70,000 eukaryotic species in Britain and Ireland \cite{}. In parallel, other international consortiums have initiated projects with similar aspirations for insects \cite{}, vertebrates \cite{}, invertebrates \cite{} and all of life \cite{}. The DToL project, currently, uses CCS reads for contig generation, Hi-C reads to order and orient contigs, and a Hi-C contact matrix to manually inspect and correct chromosome-length scaffolds. The DToL project regularly updates their primary sequencing and mapping technologies and assembly, purging and scaffolding algorithms to reflect the advances in the field. At the time of writing, the DToL project has sequenced approximately 800 species, completed the assemblies of approximately 500 species, and made available to the public the raw data and reference genomes \cite{}. 

\section{Thesis objectives}

The history of science is riddled with examples where theory, technology, and serendipitous discovery drive science. The advent of Illumina short reads and continued decrease in per-base sequencing cost have accelerated our understanding of human evolution and migration patterns\cite{}, identification of pathogenic mutations in patients with Mendelian diseases\cite{}, the analysis of driver mutation and transcriptomic landscape in thousands of cancer genomes\cite{}. 

The inability to generate contiguous and complete reference genomes, however, with Illumina short reads\cite{} and the prohibitively expensive cost of BAC clone library preparation and hierarchical shotgun sequencing have thwarted our efforts to understand genetic variation in non-model organisms\cite{}.

High-throughput and high-accuracy single-molecule sequencing technologies\cite{Wenger2019-pw} overcome the limitations of the Illumina platform and propel us towards the third wave of genomic revolution where each individual will be able to have their complete and haplotype-phased genome sequence, where the construction of the most complex and repetitive genomes will be possible and where the reference genomes of all organisms will be available to the scientific community. 

The DToL project, for example, has generated an extraordinary public resource that comprises CCS reads, linked reads, Hi-C reads, high-quality chromosome-length scaffolds, and associated gene annotations. Comparative genomics in linear and three-dimensional space and population genetic studies with the newly assembled reference genomes will undoubtedly enhance our understanding of the process of speciation and evolution. Here, we aspired to better understand the mutational process operational in each species.

To determine the germline and somatic mutational process across the Tree of Life, we considered the following:

\begin{enumerate}
\item Based on the similarities between the duplex\cite{Schmitt2012-yr} and CCS library sequencing \cite{Travers2010-sx} principles, we hypothesised that CCS reads might have sufficient base accuracy for ultra-rare somatic mutation and potentially single molecule somatic mutation detection.
\item CCS reads are reported to have a predicted accuracy above Q20, but their base accuracies have not been independently examined. 
\item Somatic mutation detection algorithms need to distinguish somatic mutations from germline mutations in addition to sequencing, alignment and systematic bioinformatic errors.
\item Using samples with known ongoing somatic mutational processes and mutational signature analysis, we can demonstrate that CCS reads have sufficient or insufficient base accuracy for single molecule somatic mutation detection and determine the parameters that influence sensitivity and specificity.
\item If the sample in question has either high mutation rate or high mutation burden, the expected and the correct mutational spectrum will be observable from the validation and test data sets, respectively. 
\end{enumerate} 

In short, we aimed to measure the CCS error rate, assess whether CCS bases have sufficient base accuracy for single molecule somatic mutation detection, develop a method to detect somatic mutations where a single read alignment supports the mismatch between the sample and the reference genome and apply the method to understand germline and somatic mutational processes across the Tree of Life. 

%% enhancers, promoters, ribosome profiling, 3D genome structure 
%% collapsed repeats

%% The reads produced from the Illumina platform, however, are shorter (150bp) than repetitive sequences present in the human reference genome and as a result, many of the reads cannot be mapped to the reference genome with confidence and limits the genetic analysis to regions where reads can be placed confidently. 


%% \section{Genome sequence, genetic variation and sources of mutation}
%% \section{Genetic variation and sources of mutations}
%% \section{Germline and somatic mutation}

%%
%% first cell

%% fertilised egg
%% de novo mutations, SBS1 and SBS5
%% meiotic recombination as sources of mutations
%% \section{Genetic variation and Natural Selection}
%% \subsection{Germline mutations}
%% \subsection{Mosaic mutations}
%% \subsubsection{De novo mutations}
%% \subsubsection{Gene conversions}
%% \subsubsection{Crossovers}
%% \subsection{Somatic mutations}
%% \subsection{Mutational signatures and mutational processes}
%% darwin pondered the unit of inheritance (the physical material and the mechanism responsible for changing the physical material)
%% enodgenous and exogenous somatic mutation
%% DNA damage, repair, fixation 
%% envrionment
%% DNA polymerase infidelity, germline mutations
%% importance of somatic mutation detection, lineage tracing, driver mutations
%% a harsh environment, insult to the DNA, necessary to repair DNA damage
%% type of DNA damage: single-base substitution, 
%% what is universal about DNA? codon, degenerate, universal, 64 codons, stop-codon, start-codons
%% first protein
%% first riboenzyme?
%% first unicellular organism %% first in the sea
%% first lipid-bilayer that separates order from disorder, control of passage of molecules across a semi-permeable membrane  
%% fusions, meiotic recombination, plant recombination?
%% non-hologous end joining
%% transcripion-coupled repair
%% Selection Pressure & Natural Selection & Survival of the fittest
%% deleterious, postivie,
%% linked by DNA
%% entropy to submission
%% Scientists still have not figured out how the first unicellar organism  has arisen

%% Complexity that 
%% DNA replication, DNA polymerase fidelity, DNA polymerase error rate, as a source of first mutations
%% first multicellular-organism
%% DNA nicks, DNA double-strand breaks, cyclo-butane dimer, UV light, chemicals
%% different types of DNA polymerases, redundancies
%% Oswald Avery: amino acids, greater number of combinations, genetic sequence as the transforming substance
%% Rosalind, Watson: Structure of DNA
%% what happened from the discovery of the structure of the DNA to the human genome project?

%% in humans
%% c-elegans? other species?
%% The Tree of Life is connected through genetic sequence
%% DNA is the puzzle that links us all
%% since inception, birth, somatic mutations starts to accumualte
%% fertilsiation for most organisms
%% cellcular division for unicellular organisms
%% depending on the timing and the type of tissue in which the somatic mutations occur somatic mutations are inherited to the daughter cells or the next generation
%% some mutations result in speciation
%% some mutations lead to survival of fittest
%% some mutations have a large consequence, recombination, structural variations
%% the study of mutations across the Tree of Life has been limited by the cost of reference genome construction and the availiabilty of reference genomes for population genetics and for comparative genomics.
%% the cost of reference genome construction has been prohibitively high
%% the human geome project, for example, cost 3 billion dollars, a dollar per base.
%% international collaboration, multiple sequencing centers with thousands of people
%% multiple-years
%% physical-maps %% fish %% restriction-enzyme based
%% YACs
%% fosmid 50kb-200kb
%% bacterial artificial chromosome clone 100kb fragments
%% gaps, missing sequences, acrocentric chromosomes, large sections of chromosome Y
%% unplaced, unlocalised chromsomes and contigs
%% placement of contigs, scaffolding of contigs
%% Sanger di-deoxy sequencing, limited to 500bp to 1000bp
%% Solexa and Illumina sequencing by synthesis
%% de brujin graph based assemblies are short, fragmented and incomplete   
%% high-throughput, relatively high accuracy  of short-reads
%% de novo assembly quality is a function of read depth, base accuracy,  read length and complexity/repetitiveness of the target genome, %% Eric Lander
%% assemblies/genomes are abundant with sequences that are longer than Illumina read length: SINE, LINEs, repeat expansions, segmental duplications 
%% longer read length is required to trasverse the repetitive sequence and uniquely locate/place the read amongst other reads, reads are collapsed into contigs in the face of high repetitive sequences
%% scaffolding technologies: mate-pair sequencing with longer-read inserts insufficient and not scalable
%% assembly and comparative genomics didn't improve in the last decade
%% cost was high, and the effort did not yield sufficiently meaningful assembly results
%% initially Single-molecule sequencing from Oxford Nanopore Technologies and Pacific Biosciences were also inaccurate and the read length were not magnitude of orders longer, low throughput
%% continuous long read sequencing from Pacific Biosciences, 10-15kb in read length with 10-15% error rate, the errors were thought to be random, free of amplification bias
%% sufficiently long enough to trasverse repeats, however not sufficient to distinguish between unique copies of segmental duplications
%% used to reconstruct missing sequences in the human reference genome %% eichler
%% updates in the human reference genome %% tina
%% CHM1 and CHM13 seuqencing to identify structural variations
%% pathogenic mutations/repeat expansions
%% ONT for chrY centromere sequencing
%% alpha-satelitte expansion
%% usefulness of haploid genomes
%% T2T consortium, for example, recently, completed the end-to-end assembly of CHM13 genome  
%% high-throughput chromatin conformation capture sequencing (Hi-C), similar to mate-pair sequencing in concept, but across the whole-genome
%% 3C job-dekker, loops, configurations
%% originally used to study the three-dimensional genome configuration
%% chromosomes self-aggregate
%% end of one chromosome is in more contact with the end of the same chromosome than another chromosome
%% what about contacts between paternal and maternal haplotype of the same chromosome?
%% sequences in close proximity are in contact with each other more
%% conctact matrix can be used to discern correct assemblies from misassemblies
%% order and orient contigs %% matrix inversion, %% techniques from linear algebra
%% manually curate scaffolding and correct assemblies  
%% studying the genomes from the Tree of Life provides snapshots of environments that the genomes were under through space and time
%% events that might have spurred natural selection, speciation and radiation
%% timed the emergence of species, but never timed the emergence of unique somatic mutational processes across time and space
%% assembly: assumption: haploid genome
%% Pacific Biosciences circular consensus sequencing, increase in the number of ZMWs per SMRTcell from 1 million to 8 million, circular consensus sequencing instead of continuous long read sequencing
%% increase in DNA polymerase processivity, continuous long-read sequencing perhaps once or twice per molecule, circular consensu sequencing: 8 to 16 times per molecule
%% because the errors are thought to be random, highly accurate circular consensus sequence generation is possible
%% estimated to have accuracy between Q20 and Q30
%% assemblies produced from PacBio CCS reads have accuracy between Q50 and Q60.
%% massive incerase in the contiguity and completeness and assembly of the genome
%% time to complete the genome
%% cost to complete the genome
%% thousands of scientists to handful of scientists
%% except for the most complex genome
%% significant upgrade in the quality of the genome compared to that produced from short reads
%% also comparable to that produced through the human genome project
%% or small organisms or unicellular organims with limited DNA %% low-input protocol makes this possible albeit with errors introduced during PCR amplification %% bias towards sampling of reads or amplification of dna molecules before library preparation  
%% the number of eukaryotic species sequenced and assemblies with PacBio sequencing increased dramatically since the introduction of long-read sequencing
%% uncovering the evolutionary history of these species

%% Methods to study somatic mutations in cancer
%% the reasons to study cancer
%% somatic mtuational processes in cancer
%% mutational patterns, mutational signatures
%% tumour and matched normal
%% technical limitations of short reads
%% sub-cloncal
%% minute fraction


%% Methods to study somatic mutations in normal tissues
%% single-cell PCR amplification and sequencing
%% single-cell clone expansion and sequencing
%% duplex sequencing, nanorate sequencing
%% laser-capture and microdissection and sequencing of clonal tissues
%% driver mutations
%% drug resistance development
%% evolutionary history of cancers
%% developmental biology
%% lineage-tracing 

%% Wellcome Trust Sanger Institute has initiated the Darwin Tree of Life project to sequence approximately ~66,000 eukaryotic species in the and the primary mode of sequencing is CCS sequencing, hi-c sequencing
%% to sequence and assemble the samples with CCS sequencing, scaffold the samples with Hi-C reads and to curate the scaffolded assemblies through manual inspection of the contact matrix  
%% We and others have hypothesized the potential for CCS sequencing for somatic mutation detection
%% Nanorate sequencing, blunt-end restriction enzyme digestion, DNA nicks, dideoxy nucleic acid, DNA damage during sonication %% to preserve the native DNA molecule and to sequence the DNA molecule  


%% We noticed the high simliarity between duplex sequencing and CCS sequencing and hypothesized that CCS sequencing might have sufficient base accuracy for single molecule somatic mutation detection, if we can distinguish highly accurate bases from that resulting from library errors, alignment errors and sequencing errors and systematic errors. artefacts that cannot be removed 
%% Other mammalian species with shorter life span have higher somatic mutation rate such that at the terminal stages of life, the species in question have same mutation burden at the time of death
%% Peto's paradox
%% resequencing studies have enabled the identification of germline mutational process, somatic mutational process in humans
%% the study of other species have been limited to date
%% c-elegans? %% what are other species?

%% if our hypothesis is true, we conjectured that we will able to detect somatic mutations across the Tree of Life, reveal somatic mutational processes active in the species, time the emergence of somatic mutational processes and attribute the contribution of somatic mutational processes to the germline mutational process, %% environmental mutagenesis

%% in Chatper 2, we demonstrate that PacBio CCS base accuracy is sufficiently accurate to call and study single molecule somatic single-base-substitution across species
%% sequence samples with a single dominant somatic mutational process 
%% know the mutational signature or have gold-standard mutational signature for the sample generated from single-cell clone expansion and sequencing
%% 
%% somatic mutation detection from a single read alignment to the reference genome
%% if we were to call every mismatch between the read and reference genome, we will be able to call all somatic mutations at the cost of high false positive rate
%% Oxidative DNA damage
%% typically requires a normal sample to distinguish between germline and somatic mutations
%% typically requires multiple reads to suppport the somatic single base substitution  
%% VCF file produced from somatic mutation callers are the sum of library errors, systematic errors, sequencing errors, alignment errors, %% reference bias?
%% unresolved errors
%% if we are able call somatic mutations from a single read alignment to the reference genome, we are not only able to reduce the cost of sequencing, but also do germline mutation calling from reduced read depth
%% 30X sequence coverage required to call heterozygous mutations %% reference
%% problems with PacBio CCS sequencing: incomplete removal of adapter sequences, chimeric sequences resulting from problems with adapater sequence calling, fragmer and concatmer
%% reads significantly shorter and longer than the read-of-insert length
%% empirically estimate the PacBio CCS base accuracy
%% PacBio CCS base accuracy has not been measured yet, PacBio CCS base also cannot be measured with exisiting sequencing technologies with lower base accuracy


%% in Chapter 3, confirm that our method is applicable to other eukaryotic species, we use the newly developed method to study somatic mutational processes across the ~400 eukaryotic species sequenced the Darwin Tree of Life project, attempt to understand both the germline and somatic mutational processes across species, identify potential sources of environemtnal mutagenesis
%% phorcus lineatus: age
%% insects: life cycle of insects (choleoptera)
%% mutation burden of insects with metamorphosis and without metamorphosis
%% germline mutational process
%% somatic mutational process
%% environmental mutagenesis

%% in Chapter 4 and 5, we use the unique combination of long read length and base accuracy of PacBio CCS reads to study both meiotic and mitotic recombniation, respectively.
%% in Chapter 2 and Chapter 3, we demonstrate that PacBio CCS reads have sufficient read length and base accuracy for single molecule somatic single-base substitution agnostic of clonality and species.
%% to explore the unexplored phenomena of meiotic recombniation through Sperm PacBio CCS sequencing
%% diffences to previous attempts to understand meiotic recombination through trio sequencing and sperm-typing
%% gene conversions requires the detection of chimeric dna molecules with both maternal and paternal sequences
%% meiotic event generates 2 recombinant products and 2 wild type molecules
%% crossover leads to the generation of dna molecule with a stretch of paternal hetsnps followed by a stretch of maternal hetsnps and vice versa
%% gene conversion leads to the generation of a dna molecule where paternal hetsnps is flanked by maternal hetsnps (and vice versa)
%% complex recombinant product with resulting from both crossover and gene conversion
%% on average, there 1 SNP per 1000bp
%% requires long-range PCR products to detect 
%% hotspots
%% coldspots
%% meiotic recombination product requires reads that can span multiple hetsnps and requires sufficient base accuracy to determine that hetsnp switch is a result of a biological event rather than a sequencing error.
%% in addition, meiotic recombination can be a source of mutagenic event
%% PacBio CCS reads have sufficient base accuracy to detect single molecule recombination events and associated mutations
%% recombniationi might not be a perfect/clean
%% mutational process that generates de novo single-base substitution seems to be driven by clock-like somatic mutational processes (SBS1 and SBS5)
%% mitotic gene conversion can be a source of oncogenic mechanism in somatic cells
%% simliar to meiotic recombination, products from mitotic recombination cannot be detected with short reads due to the technical limiations of the Illumina platform
%% sequenced Bloom syndrome patient samples with defects in DNA double-strand break damage repair process
%% known to have gene conersions or loss of heterozygous caused by gene conversions
%% perfect sample to assess the differences in mitotic and meiotic recombniation and gene conversions
%% mitotic gene conversions are thought to be longer in length


%% in Chapter 6, 
%% the benefits of PacBio CCS sequencing
%% the last DNA sequencing platform
%% requires significantly less sequencing coverage than short reads to detect  the same number of mutations
%% can detect small SNPs, indels, structural variations, 5mC from the same platform
%% with the development of himut, CCS reads can be also used to detect somatic mutations, gene conversion and crossovers from the same sample.
%% potentially other base modifications caused by environmental exposure, chemotherapeutics in the future
%% Moore's law: the number of transisitors per semiconductor has doubled, the distance at which the electrons has to be moved has shorteneed
%% the cost of sequencing per base was decreasing at a faster speed than Moore's law and many has anticipated that we might have a $100 genome, if the development had continued
%% stagnation in development, and Illumina monopoly status, financialisation, stock buybacks instead of research and develompent
%% increase in the number of ZMWs per SMRTcell, PacBio has achieved 8-fold improvement in throughput
%% increase in the read-of-insert length, doubling, stabiltiy of the circular template molecule
%% direct-engineering, directed-natural selection
%% increase in DNA polymerase processivity can increase either the read-of-insert length or the base accuracy of the same read-of-insert length
%% improvement in HMW DNA extraction, from the smallest organism
%% past Illumina platform generation has also required high DNA concentration 
%% improvements in circular consensus sequence calling process can lead to the better discernment of adapter sequences from 
%% PacBio CCS sequencing offers an alternative method for DNA sequencing with potential to improve throughput and base accuracy at a faster rate than that from Illumina unless Illumina profit margin compresses
%% PacBio CCS sequencing will be cheaper, more accurate, have higher throughput than Illumina sequencing
%% Illumina might compete in terms of price, but the wealth of information that is delivered from PacBio will be immense %% adoption curve
%% the cumulative improvement will us to better understand all of life


%% \subsection{The cost of Reference genomes as a bottleneck}

%********************************** %Second Section  *************************************


%********************************** % Third Section  *************************************

%% \section{Thesis aims}

