%!TEX root = ../thesis.tex
%*******************************************************************************
%*********************************** First Chapter *****************************
%*******************************************************************************

\chapter{from Last Universal Common Ancestor to the Darwin Tree of Life project}  %Title of the First Chapter

\ifpdf
    \graphicspath{{Chapter1/Figs/Raster/}{Chapter1/Figs/PDF/}{Chapter1/Figs/}}
\else
    \graphicspath{{Chapter1/Figs/Vector/}{Chapter1/Figs/}}
\fi


%********************************** %First Section  **************************************

\section{Origins of Life}

We cannot understate the significance that we can study physics and chemistry anywhere in the universe, but we can only study biology on planet Earth. We search for signs of life elsewhere in the universe, but we have yet to succeed in this endeavour. Astrobiologists search for conditions favourable to the emergence of life, but these conditions do not necessarily guarantee the existence and perseverance of life outside of our solar system. We must, hence, assume that the conditions and characteristics of life can only be dissected and described here on Earth and what distinguishes the inanimiate from animate can be only understood here on Earth. 


\subsection{Prebiotic Earth}
panspermia

\subsection{The Garden of Eden}

Hydrothermal vent

\subsection{RNA world}

\subsection{The emergence and evolution of life on Earth}

Ribozyme
Endosymbiosis
Phagocytosis
energy: mtDNA
archaea
eukaryotes: organelles
prokaryotes
oxidation and photosynthesis
sex: meiotic recombination
Oxidation

\section{From Darwin and Mendel to the Human Genome Project}

\subsection{Cell Theory}

Microscopes, organelles
Pasteur


\subsection{Genetics: the study of inheritance (heredity and variation)}
Charles Darwin
Thomas Hunt Moragn
Bateson
Fisher



\subsection{Gene: the unit of inheritance, the unit of and the unit of }

X-ray irradiation, fruitflies
What is Life? aperiodic crystal
Oswald Avery
%% "Let there be light" \begin{flushright} Genesis 1:3 \end{flushright}

%% According to Aristotle's theory of heredity, males are responsible for providing the "active element" that gives life to the offspring and makes sure that it is a male or female, and females provide the nutrients to the offspring.
%% https://thelampstand.com.au/the-amazing-living-cell-a-model-for-christs-ecclesia-wrong-part-2/
%% aperiodic crystal
%% reverse transcription
search for the unit of inheritance
discovery of DNA structure
central dogma

\subsection{Central Dogma: from DNA to protein}

\subsection{Sanger sequencing}
ABI capillary sequencing

\subsection{Human Genome Project}

\subsubsection{Genome assembly}

e start of time, entropy has been increasing following the second law of thermodynamics and biological systems have emerged to reduce or maintain entropy using energy. Phospholipid permeable-membrane was the first spontaenous invention that separated order from disorder and allowed for the movement of molecules between the extracelular and intercellular environemtnt and for the emergence of primordial cell. It is uncertain whether the first cell had both the capacity to recplicate itself or whether had the capacity to catalyze chemical reaction first. In a prebiotic environment, amino acids can be created in a reducing environment if sufficient energy in the form of ioniznig radiation, ultra-violet light, is introduced into a gaseous atmosphere containing methane, ..., ... and ... [ref] and nucleotide bases are thought to be harder to spotaneously create in a prebiotic envrionment [ref]. Despite the uncertainity in how the first cell arised, the first prokaryotic organism is thought to have arise ~XX billion years ago and the first eukaryotic organism is thought to have arisen approximately 2 billion years ago [ref]. Once the first cell was created, selection pressure and natural selection acted upon these cells to create the first multicelullar organism and these muticellular organisms evolved to create multiple different species that is best adapated to the environment surrounding them. Mutations play a central role in creating new innovations that allows for individual species to better adapt to the environment and to produce progenitors that inherit the mutations. 

It is now widely accepted truth that DNA is the unit of inheritance and that DNA has a double-helix structure and that the structure of the DNA drives many of the important chemical reactions in the cells such DNA replication and transcription. In addition, sequencing technologies has become cheap enough such that clinical sequencing is routine enough to be able to detect the mutations that is responsible for disease and to understand the mutations that confer selective growth advantage to cancer genomes and amazingly, the cost of sequencing is still decreasing and new sequencing technologies are emerging to differentiate itself from short reads produced from next-generation sequencing platform. These widely accepted truth, however, were only enabled by giants who reimagined what was possible and who were willing to against the norm.  

We must have wondered about the physical material that is responsible for the unit of inheritance from ancient times [ref]. [Greeks, Romans, Bible], Gregor Mendel is thought to be the father of modern genetics and provided the theoretical framework for the study of genetics with his famous experiment where the studied he inheritance of Peas's traits to their descenents in 1866X.  Mendel carefully cross-breeded peas with different traits to discover that traits were inherited with a fixed ratio, also known as Mendelian ratio, and how certain traits are governed by dominant and recessive alelles. His experiment revealed how the physical material that is responsible for unit of inheritance must be separated into gametesand randomly united during fertilisation to determine the phenotype of the progenies and that the factors responsible for the phenotypic differences must be located independent of each other. These two rules are referred principle of segregation and principel of independent assortment.

Amino acids were initially proposed as the physical material responsible for inheritance as the number of amino acids and different varieties of protiens that could be created from different combinations of amino acould could potentialyl explain the complexity of a liviing organism and DNA was thought to be too simple to be able to encode the complexity of a living organism. It was not until the Oswald Avery's experiment in XXXX that demonstrated the DNA to be the physical material responsible for the transformation of R-strain bacteria to S-strain bacteria and despite the evidence, DNA was not believed to be physical material for unit of inheritance. The next race started with the aim of discoverying the sturcture of the DNA and there were many potential protagonists who could have discovered the structure of the DNA, but James Watson and Francis Crick, then post-doctoral fellow and PhD student at the laboratory of molecular biology, respectively, were the first to the race in 1954 [ref]. Despite the initial skepticism of how DNA could be the unit of inheritance and how DNA could be responsible for the complexity of an organism, the mechanisms of the central dogma was slowly revealed. Series of discoveries following the discovery of the structure of the DNA has cemented the importance of DNA as the central unit responsible for directing cellular behaviours and determining phenotypes and encoding the software to produce proteins, the hardware that is responsible for catalyzing chemical reactions within the cell. Despite their simplicy, methods for DNA sequencing was designed later than that for amino acid sequencing. Frederick Sanger and Walter Gilbert cames with Sanger dideoxy sequencing and Maxam-Gilbert sequencing, respectively, to determine the nucleotide monomor that consistitutes the given nuclecic acid. Sanger was able to determine the genetic sequence of XXXX and XXXX using Sanger dideoxy sequencing for the first time. The Sanger dideoxy sequencing was more amenable to sequencing at scale and was adopted for the Human Genome Project (HGP) as the primary sequencing instrument and Sanger reads produced from ABI had an average read length of 500bp to 1000bp and had an average base accuracy between Q20 and Q50. 

The Human Genome Project was initiated to sequence and assemble the human reference genome that would standardise the genetics and genomics studies to a single reference genome. There were two approaches towards the human reference genome construction: one was the hiearchical shotgun sequencing and assemgbly strategy and the other was whole-genome shotgun sequencing and assembly approach. The human reference genome constructed from the former approach is still the human reference genome used in most genetic and genomics studies and is the bedrock of genomic medicine revolution [ref]. The availability of the human reference genome together with sequencing-by-synthesis appraoch from Solexa, now Illumina, revolutionsed the field of human genetics and enabled population-scale studies of genetic diseases and cancers [ref]. Population-structure, human history, discovery of somatic mutations that confer selective growth advantage to the tumour cell, the identification of mutations that leads to genetic diseases. In addition, scientists have develooped clever ways to modify library protocol upstream of Illumina adapter ligation to enable the study of epigenomes, base modifiations, transcriptome of bulk tissue and more, recently, the advent of high-throughput chromatin conformation capture sequencing has enabled the study of the three-dimensional configuration of the genome and the nucleotide sequences are organised into regular repeating patterns. 
%% single cell %% why was hiearchical shotgun stratey more accurate?

\section{The Genomics Revolution}

%% single cell %% why was hiearchical shotgun stratey more accurate?

\subsection{Illumina sequencing}
The technical limitations of Illumina sequencing (base accuracy and short read length), however, has been the bottleneck for improving rare genetic disease diagnostics yield, detecting rare somatic mutations and constructing high-quality reference genomes for non-human species. De novo assembly of other species, previously, have been attempted using de brugjin graph based de novo assembly algorithms with short reads, but assemblies produced from short reads were highly fragmented and incomplete. In addition, scaffolding strategies often did not provide sufficient long-range information to produce chromosome-level pseudomolecules and as a result, these assemblies provided incomplete information for comparative genomics purposes. Hence, assemblies produced from short reads often have collapsed repeats or contigs that cannot be placed accurately. To construct complete assemblies, reads need to be longer than the repeats of the target genome such that the reads can traverse the repetitive regions and optimally have unique sequences flanking the repetitive sequences such that the read can be placed in the assembly graph unambiguously. Not all repetitive sequences are actually repetitive. There are unique class of repeats called segmental duplications, which doesn't have a classical repetitive sequence, has a unique sequence, but is duplicated across the many parts of the genome and are thought to be important in driving evolution and these segmental duplications are typically defined as sections greater than 1kb with sequence simliarity above 90\% to other regions of the genome. To distinguish segmental duplications from one another, reads also need to have high base accuracy to be able to distinguish closest segmental duplications from one another. Long-reads from third-generation sequencing technologies such as Oxford Nanopore Technologies (ONT) and Pacific Biosciences (PacBio) provide an alternative towards improving the rare genetic diagnostics yield and improving the reference genome qualities in terms of both completeness and contigutiy. Long-reads produced from third-generation sequencing platforms were orders-of-magnitude longer than that from the Illumina platform, but had a much higher error rate; 10-15\% error rate for continuous long reads (CLR) from PacBio and 20-35\% error rate for ONT reads. Because of these high error rates, higher sequencing costs (lower yield per dollar) and insufficient improvement in read length, these platforms had limited use except for rare cases for real-time monitoring of ... and de novo assembly of plants and animal genomes..., and detection of pathogenic mutations that could not be detected with short reads [ref, ref]. Despite high error rate, the longer read length enabled the detection of structural variations that could not be previously detected with short reads, doubled the number of structural variations that can be detected from a typical human genome compared to the human reference genome. The longer read length allowed for the de novo assembly of BAC clones to hierarchically assemble missing sequences, also known as gaps, in the human reference genome, which have been problematic to assemble before and reveal human-specific gene duplications.   

these companies have improved their library preparation protocol and base callers to improve the base accuracy. PacBio, for example, came up with circular consensus sequencing protocl in 2014, but this protocol had limited use commercially until 2018 because of insufficient DNA polymerase processivitively.   

\subsubsection{Liquid Biopsy}

\subsubsection{Clinical sequencing}

\subsubsection{Single-cell sequencing}

linked-reads

\subsubsection{High-throughput chromatin conformation capture sequencing}

Job Dekker
Erez Liberman Aiden
CTCF, Cohesin, WAPL

\subsubsection{Strand-seq}

\subsubsection{Genome engineering}

\subsection{Single molecule sequencing}

\subsubsection{Oxford Nanopore sequencing}

\subsubsection{Pacific Biosciences single molecule real time sequencing}



\section{The Darwin Tree of Life Project}

The advent of high-throughput long-read sequencing and genome mapping technologies \cite{}, improvements in base accuracy of long reads \cite{} and development of algorithms that take advantage of the longer read length and long-range genomic interactions \cite{} has brought new enthusiasm to sequence and assemble high-quality reference genomes \cite {}. 

The Darwin Tree of Life (DToL) project is an ambitious project that aspires to construct chromosome-length scaffolds for 70, 000 eukaryotic species in Britain and Ireland \cite{}. In parallel, other international consortiums has initiated projects with similar aspirations for insects \cite{}, vertebrates \cite{}, invertebrates \cite{} and all of life \cite{}. The DToL project, currently, uses CCS reads for contig generation, Hi-C reads to order and orient contigs, and Hi-C contact matrix to manually inspect and correct chromosome-length scaffolds. We would like to highlight that the DToL project regularly updates their primary sequencing and mapping technologies and assembly, purging and scaffolding algorithms to reflect the advancements in the field. At the time of writing, the DToL project has sequenced approximately 800 species, completed the assemblies of approximately 500 species, and raw data and reference genomes have been made available to the public \cite{}. 

\section{Thesis objectives}

The history of science is riddled with examples where theory, technology, and serendipitous discovery drives science. The advent of Illumina short reads and continued decrease in per base sequencing cost has accelerated our understanding of human evolution and migration patterns \cite{}, identification of pathogenic mutations in patients with Mendelian diseases \cite{}, the analysis of driver mutation and transcriptomic landscape in thousands of cancer genomes \cite{}. 

The inability to generate contiguous and complete reference genomes, however, with Illumina short reads and the prohibitively expensive cost of BAC clone library preparation and hierarchical shotgun sequencing has thwarted our efforts to understand genetic variation in non-model organisms \cite{}.

High-throughput and high-accuracy single-molecule sequencing technologies \cite{} overcome the limitations of the Illumina platform and propel us towards the third wave of genomic revolution where each individual will be able to have their complete and haplotype phased genome sequence, where the construction of the most complex and repetitive genomes will be possible and where the reference genomes of all organisms will be available to the scientific community. 

The DToL project, for example, has generated an extraordinary public resource that comprises CCS reads, linked reads, Hi-C reads, high-quality chromosome-length scaffolds, and associated gene annotations. Comparative genomics in linear and three-dimensional space and population genetic studies with the newly assembled reference genomes will undoubtedly enhance our understanding of the process of speciation and evolution. Here, we instead aspired to better understand the mutational process operational in each species.

To determine the germline and somatic mutational process across the Tree of Life, we considered the following:

\begin{enumerate}
\item Based on the similarities between the duplex \cite{} and CCS library sequencing \cite{}, we hypothesised that CCS reads might have sufficient base accuracy for ultra-rare somatic mutation and potentially single molecule somatic mutation detection.
\item CCS reads are reported to have a predicted accuracy above Q20, but their base accuracies have not been independently examined. 
\item Somatic mutation detection algorithm needs to distinguish somatic mutations from germline mutations, in addition to, sequencing, alignment and systematic bioinformatic errors. We, unfortunately, cannot differentiate somatic mutations from library errors unless there are upstream modifications to the library preparation protocol. 
\item Using samples with single ongoing somatic mutational process and mutational signature analysis, we can demonstrate that CCS reads have sufficient or insufficient base accuracy for single molecule somatic mutation detection and determine the parameters that influence sensitivity and specificity.
\item If the sample in question has either high mutation rate or high mutation burden, the expected and the correct mutational spectrum will be observable from the validation and test data sets, respectively. 
\end{enumerate} 

In short, we aimed to measure the CCS error rate, assess whether CCS bases have sufficient base accuracy for single molecule somatic mutation detection, develop a method to detect somatic mutations where a single read alignment supports the mismatch between the sample and the reference genome and apply the method to understand germline and somatic mutational processes across the Tree of Life. 



%% enhancers, promoters, ribosome profiling, 3D genome structure 
%% collapsed repeats

%% The reads produced from the Illumina platform, however, are shorter (150bp) than repetitive sequences present in the human reference genome and as a result, many of the reads cannot be mapped to the reference genome with confidence and limits the genetic analysis to regions where reads can be placed confidently. 


%% \section{Genome sequence, genetic variation and sources of mutation}
%% \section{Genetic variation and sources of mutations}
%% \section{Germline and somatic mutation}

%%
%% first cell

%% fertilised egg
%% de novo mutations, SBS1 and SBS5
%% meiotic recombination as sources of mutations
%% \section{Genetic variation and Natural Selection}
%% \subsection{Germline mutations}
%% \subsection{Mosaic mutations}
%% \subsubsection{De novo mutations}
%% \subsubsection{Gene conversions}
%% \subsubsection{Crossovers}
%% \subsection{Somatic mutations}
%% \subsection{Mutational signatures and mutational processes}
%% darwin pondered the unit of inheritance (the physical material and the mechanism responsible for changing the physical material)
%% enodgenous and exogenous somatic mutation
%% DNA damage, repair, fixation 
%% envrionment
%% DNA polymerase infidelity, germline mutations
%% importance of somatic mutation detection, lineage tracing, driver mutations
%% a harsh environment, insult to the DNA, necessary to repair DNA damage
%% type of DNA damage: single-base substitution, 
%% what is universal about DNA? codon, degenerate, universal, 64 codons, stop-codon, start-codons
%% first protein
%% first riboenzyme?
%% first unicellular organism %% first in the sea
%% first lipid-bilayer that separates order from disorder, control of passage of molecules across a semi-permeable membrane  
%% fusions, meiotic recombination, plant recombination?
%% non-hologous end joining
%% transcripion-coupled repair
%% Selection Pressure & Natural Selection & Survival of the fittest
%% deleterious, postivie,
%% linked by DNA
%% entropy to submission
%% Scientists still have not figured out how the first unicellar organism  has arisen

%% Complexity that 
%% DNA replication, DNA polymerase fidelity, DNA polymerase error rate, as a source of first mutations
%% first multicellular-organism
%% DNA nicks, DNA double-strand breaks, cyclo-butane dimer, UV light, chemicals
%% different types of DNA polymerases, redundancies
%% Oswald Avery: amino acids, greater number of combinations, genetic sequence as the transforming substance
%% Rosalind, Watson: Structure of DNA
%% what happened from the discovery of the structure of the DNA to the human genome project?

%% in humans
%% c-elegans? other species?
%% The Tree of Life is connected through genetic sequence
%% DNA is the puzzle that links us all
%% since inception, birth, somatic mutations starts to accumualte
%% fertilsiation for most organisms
%% cellcular division for unicellular organisms
%% depending on the timing and the type of tissue in which the somatic mutations occur somatic mutations are inherited to the daughter cells or the next generation
%% some mutations result in speciation
%% some mutations lead to survival of fittest
%% some mutations have a large consequence, recombination, structural variations
%% the study of mutations across the Tree of Life has been limited by the cost of reference genome construction and the availiabilty of reference genomes for population genetics and for comparative genomics.
%% the cost of reference genome construction has been prohibitively high
%% the human geome project, for example, cost 3 billion dollars, a dollar per base.
%% international collaboration, multiple sequencing centers with thousands of people
%% multiple-years
%% physical-maps %% fish %% restriction-enzyme based
%% YACs
%% fosmid 50kb-200kb
%% bacterial artificial chromosome clone 100kb fragments
%% gaps, missing sequences, acrocentric chromosomes, large sections of chromosome Y
%% unplaced, unlocalised chromsomes and contigs
%% placement of contigs, scaffolding of contigs
%% Sanger di-deoxy sequencing, limited to 500bp to 1000bp
%% Solexa and Illumina sequencing by synthesis
%% de brujin graph based assemblies are short, fragmented and incomplete   
%% high-throughput, relatively high accuracy  of short-reads
%% de novo assembly quality is a function of read depth, base accuracy,  read length and complexity/repetitiveness of the target genome, %% Eric Lander
%% assemblies/genomes are abundant with sequences that are longer than Illumina read length: SINE, LINEs, repeat expansions, segmental duplications 
%% longer read length is required to trasverse the repetitive sequence and uniquely locate/place the read amongst other reads, reads are collapsed into contigs in the face of high repetitive sequences
%% scaffolding technologies: mate-pair sequencing with longer-read inserts insufficient and not scalable
%% assembly and comparative genomics didn't improve in the last decade
%% cost was high, and the effort did not yield sufficiently meaningful assembly results
%% initially Single-molecule sequencing from Oxford Nanopore Technologies and Pacific Biosciences were also inaccurate and the read length were not magnitude of orders longer, low throughput
%% continuous long read sequencing from Pacific Biosciences, 10-15kb in read length with 10-15% error rate, the errors were thought to be random, free of amplification bias
%% sufficiently long enough to trasverse repeats, however not sufficient to distinguish between unique copies of segmental duplications
%% used to reconstruct missing sequences in the human reference genome %% eichler
%% updates in the human reference genome %% tina
%% CHM1 and CHM13 seuqencing to identify structural variations
%% pathogenic mutations/repeat expansions
%% ONT for chrY centromere sequencing
%% alpha-satelitte expansion
%% usefulness of haploid genomes
%% T2T consortium, for example, recently, completed the end-to-end assembly of CHM13 genome  
%% high-throughput chromatin conformation capture sequencing (Hi-C), similar to mate-pair sequencing in concept, but across the whole-genome
%% 3C job-dekker, loops, configurations
%% originally used to study the three-dimensional genome configuration
%% chromosomes self-aggregate
%% end of one chromosome is in more contact with the end of the same chromosome than another chromosome
%% what about contacts between paternal and maternal haplotype of the same chromosome?
%% sequences in close proximity are in contact with each other more
%% conctact matrix can be used to discern correct assemblies from misassemblies
%% order and orient contigs %% matrix inversion, %% techniques from linear algebra
%% manually curate scaffolding and correct assemblies  
%% studying the genomes from the Tree of Life provides snapshots of environments that the genomes were under through space and time
%% events that might have spurred natural selection, speciation and radiation
%% timed the emergence of species, but never timed the emergence of unique somatic mutational processes across time and space
%% assembly: assumption: haploid genome
%% Pacific Biosciences circular consensus sequencing, increase in the number of ZMWs per SMRTcell from 1 million to 8 million, circular consensus sequencing instead of continuous long read sequencing
%% increase in DNA polymerase processivity, continuous long-read sequencing perhaps once or twice per molecule, circular consensu sequencing: 8 to 16 times per molecule
%% because the errors are thought to be random, highly accurate circular consensus sequence generation is possible
%% estimated to have accuracy between Q20 and Q30
%% assemblies produced from PacBio CCS reads have accuracy between Q50 and Q60.
%% massive incerase in the contiguity and completeness and assembly of the genome
%% time to complete the genome
%% cost to complete the genome
%% thousands of scientists to handful of scientists
%% except for the most complex genome
%% significant upgrade in the quality of the genome compared to that produced from short reads
%% also comparable to that produced through the human genome project
%% or small organisms or unicellular organims with limited DNA %% low-input protocol makes this possible albeit with errors introduced during PCR amplification %% bias towards sampling of reads or amplification of dna molecules before library preparation  
%% the number of eukaryotic species sequenced and assemblies with PacBio sequencing increased dramatically since the introduction of long-read sequencing
%% uncovering the evolutionary history of these species

%% Methods to study somatic mutations in cancer
%% the reasons to study cancer
%% somatic mtuational processes in cancer
%% mutational patterns, mutational signatures
%% tumour and matched normal
%% technical limitations of short reads
%% sub-cloncal
%% minute fraction


%% Methods to study somatic mutations in normal tissues
%% single-cell PCR amplification and sequencing
%% single-cell clone expansion and sequencing
%% duplex sequencing, nanorate sequencing
%% laser-capture and microdissection and sequencing of clonal tissues
%% driver mutations
%% drug resistance development
%% evolutionary history of cancers
%% developmental biology
%% lineage-tracing 

%% Wellcome Trust Sanger Institute has initiated the Darwin Tree of Life project to sequence approximately ~66,000 eukaryotic species in the and the primary mode of sequencing is CCS sequencing, hi-c sequencing
%% to sequence and assemble the samples with CCS sequencing, scaffold the samples with Hi-C reads and to curate the scaffolded assemblies through manual inspection of the contact matrix  
%% We and others have hypothesized the potential for CCS sequencing for somatic mutation detection
%% Nanorate sequencing, blunt-end restriction enzyme digestion, DNA nicks, dideoxy nucleic acid, DNA damage during sonication %% to preserve the native DNA molecule and to sequence the DNA molecule  


%% We noticed the high simliarity between duplex sequencing and CCS sequencing and hypothesized that CCS sequencing might have sufficient base accuracy for single molecule somatic mutation detection, if we can distinguish highly accurate bases from that resulting from library errors, alignment errors and sequencing errors and systematic errors. artefacts that cannot be removed 
%% Other mammalian species with shorter life span have higher somatic mutation rate such that at the terminal stages of life, the species in question have same mutation burden at the time of death
%% Peto's paradox
%% resequencing studies have enabled the identification of germline mutational process, somatic mutational process in humans
%% the study of other species have been limited to date
%% c-elegans? %% what are other species?

%% if our hypothesis is true, we conjectured that we will able to detect somatic mutations across the Tree of Life, reveal somatic mutational processes active in the species, time the emergence of somatic mutational processes and attribute the contribution of somatic mutational processes to the germline mutational process, %% environmental mutagenesis

%% in Chatper 2, we demonstrate that PacBio CCS base accuracy is sufficiently accurate to call and study single molecule somatic single-base-substitution across species
%% sequence samples with a single dominant somatic mutational process 
%% know the mutational signature or have gold-standard mutational signature for the sample generated from single-cell clone expansion and sequencing
%% 
%% somatic mutation detection from a single read alignment to the reference genome
%% if we were to call every mismatch between the read and reference genome, we will be able to call all somatic mutations at the cost of high false positive rate
%% Oxidative DNA damage
%% typically requires a normal sample to distinguish between germline and somatic mutations
%% typically requires multiple reads to suppport the somatic single base substitution  
%% VCF file produced from somatic mutation callers are the sum of library errors, systematic errors, sequencing errors, alignment errors, %% reference bias?
%% unresolved errors
%% if we are able call somatic mutations from a single read alignment to the reference genome, we are not only able to reduce the cost of sequencing, but also do germline mutation calling from reduced read depth
%% 30X sequence coverage required to call heterozygous mutations %% reference
%% problems with PacBio CCS sequencing: incomplete removal of adapter sequences, chimeric sequences resulting from problems with adapater sequence calling, fragmer and concatmer
%% reads significantly shorter and longer than the read-of-insert length
%% empirically estimate the PacBio CCS base accuracy
%% PacBio CCS base accuracy has not been measured yet, PacBio CCS base also cannot be measured with exisiting sequencing technologies with lower base accuracy


%% in Chapter 3, confirm that our method is applicable to other eukaryotic species, we use the newly developed method to study somatic mutational processes across the ~400 eukaryotic species sequenced the Darwin Tree of Life project, attempt to understand both the germline and somatic mutational processes across species, identify potential sources of environemtnal mutagenesis
%% phorcus lineatus: age
%% insects: life cycle of insects (choleoptera)
%% mutation burden of insects with metamorphosis and without metamorphosis
%% germline mutational process
%% somatic mutational process
%% environmental mutagenesis

%% in Chapter 4 and 5, we use the unique combination of long read length and base accuracy of PacBio CCS reads to study both meiotic and mitotic recombniation, respectively.
%% in Chapter 2 and Chapter 3, we demonstrate that PacBio CCS reads have sufficient read length and base accuracy for single molecule somatic single-base substitution agnostic of clonality and species.
%% to explore the unexplored phenomena of meiotic recombniation through Sperm PacBio CCS sequencing
%% diffences to previous attempts to understand meiotic recombination through trio sequencing and sperm-typing
%% gene conversions requires the detection of chimeric dna molecules with both maternal and paternal sequences
%% meiotic event generates 2 recombinant products and 2 wild type molecules
%% crossover leads to the generation of dna molecule with a stretch of paternal hetsnps followed by a stretch of maternal hetsnps and vice versa
%% gene conversion leads to the generation of a dna molecule where paternal hetsnps is flanked by maternal hetsnps (and vice versa)
%% complex recombinant product with resulting from both crossover and gene conversion
%% on average, there 1 SNP per 1000bp
%% requires long-range PCR products to detect 
%% hotspots
%% coldspots
%% meiotic recombination product requires reads that can span multiple hetsnps and requires sufficient base accuracy to determine that hetsnp switch is a result of a biological event rather than a sequencing error.
%% in addition, meiotic recombination can be a source of mutagenic event
%% PacBio CCS reads have sufficient base accuracy to detect single molecule recombination events and associated mutations
%% recombniationi might not be a perfect/clean
%% mutational process that generates de novo single-base substitution seems to be driven by clock-like somatic mutational processes (SBS1 and SBS5)
%% mitotic gene conversion can be a source of oncogenic mechanism in somatic cells
%% simliar to meiotic recombination, products from mitotic recombination cannot be detected with short reads due to the technical limiations of the Illumina platform
%% sequenced Bloom syndrome patient samples with defects in DNA double-strand break damage repair process
%% known to have gene conersions or loss of heterozygous caused by gene conversions
%% perfect sample to assess the differences in mitotic and meiotic recombniation and gene conversions
%% mitotic gene conversions are thought to be longer in length


%% in Chapter 6, 
%% the benefits of PacBio CCS sequencing
%% the last DNA sequencing platform
%% requires significantly less sequencing coverage than short reads to detect  the same number of mutations
%% can detect small SNPs, indels, structural variations, 5mC from the same platform
%% with the development of himut, CCS reads can be also used to detect somatic mutations, gene conversion and crossovers from the same sample.
%% potentially other base modifications caused by environmental exposure, chemotherapeutics in the future
%% Moore's law: the number of transisitors per semiconductor has doubled, the distance at which the electrons has to be moved has shorteneed
%% the cost of sequencing per base was decreasing at a faster speed than Moore's law and many has anticipated that we might have a $100 genome, if the development had continued
%% stagnation in development, and Illumina monopoly status, financialisation, stock buybacks instead of research and develompent
%% increase in the number of ZMWs per SMRTcell, PacBio has achieved 8-fold improvement in throughput
%% increase in the read-of-insert length, doubling, stabiltiy of the circular template molecule
%% direct-engineering, directed-natural selection
%% increase in DNA polymerase processivity can increase either the read-of-insert length or the base accuracy of the same read-of-insert length
%% improvement in HMW DNA extraction, from the smallest organism
%% past Illumina platform generation has also required high DNA concentration 
%% improvements in circular consensus sequence calling process can lead to the better discernment of adapter sequences from 
%% PacBio CCS sequencing offers an alternative method for DNA sequencing with potential to improve throughput and base accuracy at a faster rate than that from Illumina unless Illumina profit margin compresses
%% PacBio CCS sequencing will be cheaper, more accurate, have higher throughput than Illumina sequencing
%% Illumina might compete in terms of price, but the wealth of information that is delivered from PacBio will be immense %% adoption curve
%% the cumulative improvement will us to better understand all of life


%% \subsection{The cost of Reference genomes as a bottleneck}

%********************************** %Second Section  *************************************


%********************************** % Third Section  *************************************

%% \section{Thesis aims}

